{
 "metadata": {
  "name": "generate_fake_pops"
 }, 
 "nbformat": 2, 
 "worksheets": [
  {
   "cells": [
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "num_haplo = 20", 
      "pct_var = 5", 
      "target_cov = 8", 
      "outdir = os.path.expanduser('~/code/AFS/sim-%shaplos_exp-dist_theta%0.2f_%sxCov/' % (num_haplo,pct_var/100.0,target_cov))", 
      "print outdir"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/"
       ]
      }
     ], 
     "prompt_number": 241
    }, 
    {
     "cell_type": "code", 
     "collapsed": true, 
     "input": [
      "import re,os", 
      "from glob import glob"
     ], 
     "language": "python", 
     "outputs": [], 
     "prompt_number": 242
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "f = 'agouti-mc1r.fa'", 
      "seqd = dict([(seqid,sequence.replace('\\n','').upper()) for seqid,sequence in re.findall('>(.+?)\\n([\\w\\n]+)',open(f).read())])"
     ], 
     "language": "python", 
     "outputs": [], 
     "prompt_number": 243
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "[(k,len(v)) for k,v in seqd.items()]"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "pyout", 
       "prompt_number": 244, 
       "text": [
        "[('agouti_bac', 186368), ('mc1r_bac', 159283)]"
       ]
      }
     ], 
     "prompt_number": 244
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "e_dist = numpy.random.exponential(scale=num_haplo/20,size=100*num_haplo)", 
      "h,b,p = hist(e_dist,bins=range(num_haplo))"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "display_data", 
       "png": "iVBORw0KGgoAAAANSUhEUgAAAX4AAAD9CAYAAAC7iRw+AAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAFm5JREFUeJzt3W1sU+fBh/G/LZp0Ey5LQYRKwTDWLLZpIAm1nVGFmIix\naNKajAoBEmwiqdSllTJeViENpJJ9APEysQQtJqrqTntBaKqERrtCRiIdRjZhmxW1UnB46agC0yDN\nqIpTwkrX83yg+IHCMuyYmHBfP8lSfI5v+z7GukiOj48dtm3bAgAYw5nrCQAAxhbhBwDDEH4AMAzh\nBwDDEH4AMAzhBwDDjBj+hoYGFRYWqrS09I51P//5z+V0OnX58uXUsra2NhUXF8vn86mnpye1PJFI\nqKKiQrNmzdLGjRuzOH0AQLpGDP/q1at16NChO5afP39ehw8f1owZM1LLBgYG1N7eru7uboXDYTU3\nN6fWrV+/Xhs2bFA8HteRI0d0/PjxLG4CACAdI4a/qqpKBQUFdyxft26dtm/fftuyaDSq2tpaud1u\nVVdXy7ZtDQ0NSZJOnTqlZcuWafLkyVqyZImi0WgWNwEAkI609/H/4Q9/UFFRkebMmXPb8lgsJq/X\nm7peUlKiaDSqs2fPaurUqanlPp9Px44dG8WUAQCjMSGdG1+9elVbtmzR4cOHU8tunvHhbmd+cDgc\ndywb6QwRd7s9AOB/S+fsO2n9xv/+++/rgw8+0Ny5c/X1r39dFy5c0Lx583Tp0iUFg0GdPHkyddu+\nvj75/X49+eSTunTpUmr5yZMnVVlZOeLkuWTn8sorr+R8Dg/LheeS5/NBvqQrrfCXlpbq0qVLOnfu\nnM6dO6eioiK98847KiwsVCAQUGdnp/r7+2VZlpxOp1wulyTJ4/Fo3759Ghwc1P79+xUMBtOeKAAg\nO0YM/4oVKzR//nydPn1a06dP1+uvv37b+lt3zRQWFqqpqUk1NTV68cUX1dramlq3c+dObd++XX6/\nX1VVVXr66aezvBkAgHvlsDP5O+E+cTgcGf3ZgruzLEuhUCjX03go8FxmF89ndqXbTsIPAONcuu3k\nlA0AYBjCDwCGIfwAYBjCDwCGIfwAYBjCDwCGIfwAYBjCDwCGIfwAYBjCDwCGIfwAYBjCDwCGIfwA\nYBjCDwCGIfwAYJi0vmx9LIz2C9ddrgJduXI5S7MBgIfPA/dFLNJop8OXuQAwC1/EAgAYEeEHAMMQ\nfgAwDOEHAMMQfgAwzIjhb2hoUGFhoUpLS1PLXn75ZXm9XlVUVGjNmjUaHh5OrWtra1NxcbF8Pp96\nenpSyxOJhCoqKjRr1ixt3LjxPmwGAOBejRj+1atX69ChQ7ctW7x4sXp7e3X8+HF98skn2rt3ryRp\nYGBA7e3t6u7uVjgcVnNzc2rM+vXrtWHDBsXjcR05ckTHjx+/D5sCALgXI4a/qqpKBQUFty379re/\nLafTKafTqe985zs6cuSIJCkajaq2tlZut1vV1dWybVtDQ0OSpFOnTmnZsmWaPHmylixZomg0ep82\nBwDwv4zqk7uvvvqqnn/+eUlSLBaT1+tNrSspKVE0GtWMGTM0derU1HKfz6ff/e53eumll/7LvW6+\n5efQFxcAwE2WZcmyrIzHZxz+n/3sZ3K5XFq6dKkk3fVTY3c7/cL//nTZ5kynBABGCIVCCoVCqest\nLS1pjc8o/L/61a/U2dmp7u7u1LJgMKiurq7U9b6+Pvn9frlcLl26dCm1/OTJk6qsrMzkYQEAWZD2\n4ZyHDh3Sjh07dODAAT366KOp5YFAQJ2dnerv75dlWXI6nXK5XJIkj8ejffv2aXBwUPv371cwGMze\nFgAA0jLiSdpWrFihI0eOaHBwUIWFhWppadHWrVv16aef6vHHH5ckfetb31J7e7skqbW1Vbt371Ze\nXp46OjpUVVUl6cZv+StXrtRHH32k5cuXa+vWrXefDCdpA4C0pXuSNs7OCQDjHGfnBACMiPADgGEI\nPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGEIPwAY\nhvADgGEIPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGFGDH9DQ4MKCwtVWlqaWpZMJlVXVye32636\n+noNDQ2l1rW1tam4uFg+n089PT2p5YlEQhUVFZo1a5Y2btx4HzYDAHCvRgz/6tWrdejQoduWhcNh\nud1unTlzRkVFRdqzZ48kaWBgQO3t7eru7lY4HFZzc3NqzPr167VhwwbF43EdOXJEx48fvw+bAgC4\nFyOGv6qqSgUFBbcti8ViamxsVH5+vhoaGhSNRiVJ0WhUtbW1crvdqq6ulm3bqb8GTp06pWXLlmny\n5MlasmRJagwAYOylvY8/Ho/L4/FIkjwej2KxmKQb4fd6vanblZSUKBqN6uzZs5o6dWpquc/n07Fj\nx0Y7bwBAhiakO8C27Xu+rcPhyGD85lt+Dn1xAQDcZFmWLMvKeHza4ff7/UokEiovL1cikZDf75ck\nBYNBdXV1pW7X19cnv98vl8ulS5cupZafPHlSlZWVIzzC5nSnBABGCYVCCoVCqestLS1pjU97V08w\nGFQkEtHw8LAikUgq4oFAQJ2dnerv75dlWXI6nXK5XJJu7BLat2+fBgcHtX//fgWDwXQfFgCQJSOG\nf8WKFZo/f75Onz6t6dOn6/XXX1dTU5P6+/tVUlKif/zjH/rRj34kSSosLFRTU5Nqamr04osvqrW1\nNXU/O3fu1Pbt2+X3+1VVVaWnn376/m4VAOC/ctjp7LS/z268JzDa6TjSeh8CAMY7hyO97vHJXQAw\nDOEHAMMQfgAwDOEHAMMQfgAwDOEHAMMQfgAwDOEHAMMQfgAwDOEHAMMQfgAwDOEHAMMQfgAwDOEH\nAMMQfgAwDOEHAMMQfgAwDOEHAMMQfgAwDOEHAMMQfgAwDOEHAMMQfgAwTMbhf/XVVzV//nzNmzdP\na9askSQlk0nV1dXJ7Xarvr5eQ0NDqdu3tbWpuLhYPp9PPT09o585ACAjGYX/8uXL2rJliw4fPqx4\nPK7Tp0+rs7NT4XBYbrdbZ86cUVFRkfbs2SNJGhgYUHt7u7q7uxUOh9Xc3JzVjQAA3LuMwv+Vr3xF\ntm3r448/1vDwsK5evaqvfe1risViamxsVH5+vhoaGhSNRiVJ0WhUtbW1crvdqq6ulm3bSiaTWd0Q\nAMC9yTj84XBYM2fO1LRp0/TMM88oGAwqHo/L4/FIkjwej2KxmKQb4fd6vanxJSUlqXUAgLE1IZNB\nH374oZqamnTy5EkVFBRo6dKleuutt2Tb9j3fh8Ph+C9rNt/yc+iLCwDgJsuyZFlWxuMzCn8sFlNl\nZaWefPJJSdLSpUt19OhR+f1+JRIJlZeXK5FIyO/3S5KCwaC6urpS4/v6+lLr7rQ5kykBgDFCoZBC\noVDqektLS1rjM9rVU1VVpePHj+vy5cv697//rYMHD2rx4sUKBoOKRCIaHh5WJBJRZWWlJCkQCKiz\ns1P9/f2yLEtOp1MulyuThwYAjFJGv/E/9thj2rRpk77//e/r6tWrqq2t1cKFCxUIBLRy5UqVlJSo\noqJC27ZtkyQVFhaqqalJNTU1ysvLU0dHR1Y3AgBw7xx2Ojvm77Mb+/1HOx1HWu81AMB453Ck1z0+\nuQsAhiH8AGAYwg8AhiH8AGAYwg8AhiH8AGAYwg8AhiH8AGAYwg8AhiH8AGAYwg8AhiH8AGAYwg8A\nhiH8AGAYwg8AhiH8AGAYwg8AhiH8AGAYwg8AhiH8AGAYwg8AhiH8AGAYwg8Ahsk4/J988ol++MMf\n6pvf/KZ8Pp+i0aiSyaTq6urkdrtVX1+voaGh1O3b2tpUXFwsn8+nnp6erEweAJC+jMP/yiuvyO12\n67333tN7770nj8ejcDgst9utM2fOqKioSHv27JEkDQwMqL29Xd3d3QqHw2pubs7aBgAA0pNx+Lu6\nuvTTn/5Ujz76qCZMmKBJkyYpFoupsbFR+fn5amhoUDQalSRFo1HV1tbK7Xarurpatm0rmUxmbSMA\nAPcuo/BfuHBB165dU1NTk4LBoLZt26bh4WHF43F5PB5JksfjUSwWk3Qj/F6vNzW+pKQktQ4AMLYm\nZDLo2rVrOn36tHbs2KFFixbphRde0O9//3vZtn3P9+FwOP7Lms23/Bz64gIAuMmyLFmWlfF4h51O\nrW/h9XqVSCQkSQcPHtSvf/1rffrpp9q0aZPKy8v1t7/9TVu3btUbb7yhN998U11dXWptbZUklZWV\n6ejRo3K5XLdPxuGQlNF0br2XtP4DAoDxzuFIr3sZ7+MvLi5WNBrV559/rj/+8Y9atGiRgsGgIpGI\nhoeHFYlEVFlZKUkKBALq7OxUf3+/LMuS0+m8I/oAgLGR0a4eSdq5c6d+8IMf6Nq1a1q0aJGWL1+u\nzz//XCtXrlRJSYkqKiq0bds2SVJhYaGamppUU1OjvLw8dXR0ZG0DAADpyXhXz/3Arh4ASN+Y7eoB\nAIxPhB8ADEP4AcAwhB8ADEP4AcAwhB8ADEP4AcAwhB8ADEP4AcAwhB8ADEP4AcAwhB8ADEP4AcAw\nD2H4J8jhcGR8eeyxx3O9AQBwXz2Up2Ue3X1wWmcA4wunZQYAjIjwA4BhCD8AGIbwA4BhCD8AGIbw\nA4BhCD8AGIbwA4BhMg7/f/7zH5WXl+t73/ueJCmZTKqurk5ut1v19fUaGhpK3batrU3FxcXy+Xzq\n6ekZ/awBABnLOPytra3y+XxffNpWCofDcrvdOnPmjIqKirRnzx5J0sDAgNrb29Xd3a1wOKzm5ubs\nzBwAkJGMwn/hwgW9/fbbev7551MfE47FYmpsbFR+fr4aGhoUjUYlSdFoVLW1tXK73aqurpZt20om\nk9nbAgBAWjIK/9q1a7Vjxw45nf8/PB6Py+PxSJI8Ho9isZikG+H3er2p25WUlKTWAQDG3oR0B7z1\n1luaOnWqysvLZVlWank6Jwi6uXvo7jbf8nPoiwsA4CbLsm7rb7rSDv9f//pXHThwQG+//bauXbum\nK1euaNWqVfL7/UokEiovL1cikZDf75ckBYNBdXV1pcb39fWl1t3d5nSnBABGCYVCCoVCqestLS1p\njU97V8+WLVt0/vx5nTt3Tvv27VNNTY1+85vfKBgMKhKJaHh4WJFIRJWVlZKkQCCgzs5O9ff3y7Is\nOZ1OuVyudB8WAJAlaf/G/2U3d9s0NTVp5cqVKikpUUVFhbZt2yZJKiwsVFNTk2pqapSXl6eOjo7R\nPiQAYBT4Ipa7jH+AnhIA+J/4IhYAwIgIPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGEIPwAYhvAD\ngGEIPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGEIPwAYhvADgGEI\nPwAYhvADgGEyCv/58+e1cOFCzZ49W6FQSHv37pUkJZNJ1dXVye12q76+XkNDQ6kxbW1tKi4uls/n\nU09PT3ZmDwBIm8O2bTvdQRcvXtTFixdVVlamwcFBBQIBvfvuuwqHwzp//rx27typ9evXa+bMmfrJ\nT36igYEBLViwQH/605907tw5rV27Vu+8886dk3E4JKU9nS/fyyjvw6EMnhIAyBmHI71uZfQb/7Rp\n01RWViZJmjJlimbPnq14PK5YLKbGxkbl5+eroaFB0WhUkhSNRlVbWyu3263q6mrZtq1kMpnJQwMA\nRmnU+/jPnj2r3t5eBQIBxeNxeTweSZLH41EsFpN0I/xerzc1pqSkJLUOADC2JoxmcDKZ1LJly7Rr\n1y5NnDgxrT81buzWuZvNt/wc+uICALjJsixZlpXx+IzDf/36dT333HNatWqV6urqJEl+v1+JRELl\n5eVKJBLy+/2SpGAwqK6urtTYvr6+1Lo7bc50SgBghFAopFAolLre0tKS1viMdvXYtq3GxkY99dRT\nWrNmTWp5MBhUJBLR8PCwIpGIKisrJUmBQECdnZ3q7++XZVlyOp1yuVyZPDQAYJQyOqqnp6dHCxYs\n0Jw5c1K7bLZu3apnnnlGK1eu1IkTJ1RRUaHf/va3mjhxoiSptbVVu3fvVl5enjo6OlRVVXXnZDiq\nBwDSlu5RPRmF/34h/ACQvjE5nBMAMH4RfgAwDOEHAMOM6jj+h9OEET5jcG9crgJduXI5S/MBgOzi\nzd2sj79xHw/Q0wrgIcebuwCAERF+ADAM4QcAwxB+ADAM4QcAwxB+ADAM4QcAwxB+ADAM4QcAwxB+\nADAM4QcAwxB+ADAMZ+e8LzjDJ4AHF2fnzPr47N3HA/RPA+ABxtk5AQAjIvwAYBjCDwCGIfwAYJgx\nC/+f//xneb1eFRcXa/fu3WP1sOPYjSODRnP56lddud6Ih4ZlWbmewkOF5zO3xiz8P/7xj9XR0aGu\nri798pe/1ODg4Fg99Dj1mW4cGZT5ZXh4aOyn/ZAiVNnF85lbYxL+jz/+WJK0YMECzZgxQ4sXL1Y0\nGh2LhwYAfMmYhD8ej8vj8aSu+3w+HTt2bCwe2nDOUe4uyhv17qbHHns8108CgC95AD+5O7pPvGbn\nPh6EOWTrPkbj+qjvIZn8aNSfYn5QtLS05HoKDxWez9wZk/D7/X69/PLLqeu9vb2qra2943Z8UhUA\n7r8x2dUzadIkSTeO7Pnggw90+PBhBYPBsXhoAMCXjNmunl/84hd64YUXdP36dTU3N2vKlClj9dAA\ngFuM2eGc1dXVSiQSOnv2rJqbm29bxzH+2TVz5kzNmTNH5eXlCgQCuZ7OuNLQ0KDCwkKVlpamliWT\nSdXV1cntdqu+vl5DQxwme6/u9nxu3rxZRUVFKi8vV3l5uQ4dOpTDGY4v58+f18KFCzV79myFQiHt\n3btXUvqv0Qfik7sc459dDodDlmXpxIkTisViuZ7OuLJ69eo7QhQOh+V2u3XmzBkVFRVpz549OZrd\n+HO359PhcGjdunU6ceKETpw4cdf3+3B3jzzyiHbt2qXe3l698cYb2rRpk5LJZNqv0ZyHn2P87w/e\nKM9MVVWVCgoKblsWi8XU2Nio/Px8NTQ08PpMw92eT4nXZ6amTZumsrIySdKUKVM0e/ZsxePxtF+j\nOQ8/x/hnn8PhUE1Njerr63XgwIFcT2fcu/U16vF4+CsqC3bv3q3Kykpt27ZNyWQy19MZl86ePave\n3l4FAoG0X6M5Dz+y7y9/+Yveffddbd26VevWrdPFixdzPaVxjd9Os6upqUnnzp1TZ2en3n//fXV0\ndOR6SuNOMpnUsmXLtGvXLk2cODHt12jOw+/3+9XX15e63tvbq8rKyhzOaPx74oknJEler1fPPvus\n3nzzzRzPaHzz+/1KJBKSpEQiIb/fn+MZjW9Tp06Vw+HQpEmT9NJLL2n//v25ntK4cv36dT333HNa\ntWqV6urqJKX/Gs15+DnGP7uuXr2a+tP5ww8/VGdnJ2+ejVIwGFQkEtHw8LAikQi/mIzSP//5T0nS\nZ599pr179+q73/1ujmc0fti2rcbGRj311FNas2ZNannar1H7AWBZlu3xeOxvfOMbdmtra66nM679\n/e9/t+fOnWvPnTvXrqmpsV977bVcT2lcWb58uf3EE0/YeXl5dlFRkR2JROwrV67Yzz77rD19+nS7\nrq7OTiaTuZ7muHHz+XzkkUfsoqIi+7XXXrNXrVpll5aW2vPmzbPXrl1r/+tf/8r1NMeNo0eP2g6H\nw547d65dVlZml5WV2QcPHkz7NfpAfdk6AOD+y/muHgDA2CL8AGAYwg8AhiH8AGAYwg8AhiH8AGCY\n/wOEtXo1a/ZyugAAAABJRU5ErkJggg==\n"
      }
     ], 
     "prompt_number": 245
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "pad_h = h+20", 
      "invar = (100.0-pct_var)/pct_var", 
      "sfs = [sum(pad_h)*invar] + list(pad_h)", 
      "sfs = array(sfs)/sum(sfs)", 
      "csfs = cumsum(sfs)", 
      "print sum(sfs[1:])/float(sfs[0]), len(sfs)", 
      "plot(sfs)", 
      "ax = figure(1).axes[0]", 
      "ax.set_yscale('log')", 
      "#print sfs"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.0526315789474 20"
       ]
      }, 
      {
       "output_type": "display_data", 
       "png": "iVBORw0KGgoAAAANSUhEUgAAAXwAAAEBCAYAAAB7Wx7VAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAGTFJREFUeJzt3X9wVeWdx/HPVQFFahoJBFgCKEaTaCSxG0LjJFw0xgzI\nj5ZFzLbRkrgbjAJiB3ecbUuwWotWRVOK2Qqsyy72J1QcRmioJkF3A7Gkls3Asvy4O+yAGkBDVEDA\ns3+cTUok4r03597nnHver5k78Z4h53zmzvV7nnzPc54TsCzLEgAg4V1kOgAAID4o+ADgExR8APAJ\nCj4A+AQFHwB8goIPAD5BwQcAn6DgA4BPXBLLnZ86dUqPPPKITpw4oenTp6u0tDSWhwMAXEBMR/hv\nvfWW8vLytGLFCq1bty6WhwIAfImIC35FRYVSU1OVnZ3dY3tTU5MyMzOVnp6u2tpaSdLOnTs1duxY\nSdKJEycciAsAiFbEBX/OnDnatGnTedsXLFiguro6bdmyRcuXL9eRI0d04403av/+/ZKkgQMH9j0t\nACBqERf8wsJCJScn99jW0dEhSSoqKtLo0aNVUlKibdu2qaCgQG+//bbmzZunb37zm84kBgBExZGL\nti0tLcrIyOh+n5WVpebmZk2ZMkU/+clPnDgEAKCPYjpLJ1yBQMB0BADwpEhWuHdklk5eXp52797d\n/b6trU0TJkyIaB+WZfFy6LV48WLjGRLpxefJZ+nWV6QcKfhJSUmS7Jk6oVBI9fX1ys/Pj2gf//iP\nNWpoaHAiDgAktIaGBtXU1ET8exEX/LKyMhUUFGjPnj1KS0vT6tWrJUnLli1TVVWViouLVV1drZSU\nlIj2e8cdNQoGg5HGAQDfCQaDURX8iHv4L7/8cq/bJ06cqF27dkUcoEtrq/T1r0f96zgHJ05n8Xk6\nh8/SrIAVTSPI6RCBgHJzF+uZZ4J8IQDgSzQ0NKihoUFLliyJqJfvooJvaccO00kAwDsCgYA3C/5l\nl1n68EOpf3/TaQDAGyIt+K5ZHnnQoBq99FKD6RgA4HrRztJxzQi/vNzSxIlSZaXpNADgDZ4d4d90\nkz1TBwAQG64p+Lm54qItAMSQK9bSkaRNm2rU2hrU2bNBXXyx6TQA4F5d0zIj5ZoevmVZSk+XNmyQ\nMjNNJwIA9/NsD1+irQMAseSqgn/TTRR8AIgV1/Twa2pqdPnlQbW2Bk1HAQBXS4gefnu7lJ4uffCB\nxDNRAODCPN3DHzJE+spXpAMHTCcBgMTjqoIvcQMWAMSK6wo+M3UAIDZcV/CZqQMAseGqWTrBYFA3\n3RTUjh2SZXHhFgB6kxCzdCS70A8dKr3zjjRihOFgAOBinp6lI9mjeto6AOA81xV8iYIPALHgyoKf\nm8vUTABwmisLPiN8AHCeKwv+1VdLH34oHT1qOgkAJA7XFPyampruaUYXXSTl5NDWAYDeeP4h5p+P\nsXChNHy49PDDhkIBgMt5flpmF/r4AOAs1xZ8ZuoAgLNc29I5c0ZKSpIOH5auuMJQMABwsYRp6Vxy\niZSdbS+xAADoO9cWfIm2DgA4ydUFnwu3AOAcCj4A+IRrL9pK0qlTUnKydOyYdOmlBoIBgIt59qLt\nuXfadhkwQLr2WmnnTjOZAMCNEu5O2y4VFVJ+vlRVFedQAOBynh3hfxFm6gCAM1xf8LlwCwDOcH1L\n56OPpNRUe7nkfv3iHAwAXCzhWjqDBklpadLu3aaTAIC3ub7gS7R1AMAJFHwA8AlPFHxm6gBA37n+\noq1k32k7Zox94fYiT5yiACD2Eu6irSRdeaU0eLC0d6/pJADgXTEt+AcOHNC9996rWbNm9XlftHUA\noG9iWvCvuuoqvfjii47siwu3ANA3YRX8iooKpaamKjs7u8f2pqYmZWZmKj09XbW1tTEJ2IWCDwB9\nE1bBnzNnjjZt2nTe9gULFqiurk5btmzR8uXLdeTIEa1Zs0YLFy7UoUOHHA3a1dIxf4kZALwprIJf\nWFio5OTkHts6OjokSUVFRRo9erRKSkq0bds2lZeX69lnn9WIESN07NgxzZ07V3/605+0dOnSPgUd\nPtxeWuHgwT7tBgB865Jof7GlpUUZGRnd77OystTc3KwpU6Z0b7vyyiv1wgsvhLW/c9d2DgaDCgaD\n5/2brrbOqFHRpgYA72poaDjvuSGRiLrgOy2cxfy72jozZsQ+DwC4zecHw0uWLIno96OepZOXl6fd\n56xo1tbWpgkTJkS7u7Bw4RYAohd1wU9KSpJkz9QJhUKqr69Xfn5+1EF6e8Th51HwASDGjzgsKytT\nY2Ojjh49qqFDh+rRRx/VnDlz1NjYqLlz5+r06dOaP3++5s+fH032sG8Ptiz7jttdu+w18gHAzyJd\nWsETa+mc69ZbpUWLpNLSGIcCAJfz7Fo64bR0JNo6ABDTlk6sRXKWWrtWWr9e+vWvYxwKAFzOsyP8\ncDHCB4DouKbgh9vSSU+X3nvPXhsfAPzINy0dSbr5Zunxx6VebsYFAN9I+JaORFsHAKLhyYKfm0vB\nB4BIuabgh9vDl+wRPk+/AuBXvurhf/qp9NWvSu3t0uWXxzAYALiYL3r4/ftLmZnSn/9sOgkAeIcn\nC75EWwcAIuWagh9JD19ipg4A//JVD1+Smpul6mqKPgD/SvjVMrt88omUkmLfcdu/f4yCAYCL+eKi\nrSQNHChdfbXU1mY6CQB4g2cLvsQNWAAQCU8XfGbqAED4XFPwI52lIzFTB4A/+W6WjiR1dEh/9Vf2\nz4svjkEwAHAx31y0laSkJGnYMGnPHtNJAMD9PF3wJdo6ABAuzxd8ZuoAQHg8X/CZqQMA4fH0RVtJ\nev996dprpQ8+kAIBh4MBgIt59qJtNNMyJWnoUGnQIOnAAeczAYAb+XJaZpdp06TycmnWLAdDAYDL\neXaE3xff+Ib0+OPSyZOmkwCAeyXECN+ypDvvlIYMkX72MweDAYCL+XKEHwhIL74obd4s/epXptMA\ngDslxAi/y9tvS5MnS//xH9LYsQ4EAwAX8+UIv8tf/7X0ve9Js2dLp06ZTgMA7pJQI3zJ7ufPnCml\npUnPPefILgHAlXw9wpfsfv7KldKGDdL69abTAIB7uKbgR3vjVW+Sk6Vf/EKqqpJCIUd2CQCu4esb\nr77IM89Iv/yltHUrDzoHkHgirZ0JXfAtS5o+XUpPl55+2vHdA4BRvu/hnysQkP75n6Xf/EZ69VXT\naQDArIQe4Xf593+3l19oaZFGjYrZYQAgrhjh96KgQPrud6W77pJOnzadBgDM8MUIX5I++0y64w4p\nO1taujSmhwKAuOCi7QW0t9tPyKqrs5dgAAAvo+B/iaYme2XNt9+WRo6MyyEBICbo4X+JoiJp3jyp\nrEw6c8Z0GgCIH98VfEl65BHpssukxYtNJwGA+Il5S+eVV17Rxo0bdebMGc2dO1fjx48/P0QcWzpd\n3nvP7uevXi2VlMT10ADgCNf28N9//30tXrxYK1asOD+EgYIvSW+8If3t30p//KM0YkTcDw8AfRKz\nHn5FRYVSU1OVnZ3dY3tTU5MyMzOVnp6u2traL/z9pUuXqqqqKuxg8TBpkjR3rvStb0lnz5pOAwCx\nFXbBnzNnjjZt2nTe9gULFqiurk5btmzR8uXLdeTIEa1Zs0YLFy7UoUOHZFmWHn74YU2ePFk5OTmO\nhnfC974nXXSR9OijppMAQGxdEu4/LCwsVOhzaw13dHRIkoqKiiRJJSUl2rZtm8rLy1VeXi5Jev75\n5/X666+rs7NTe/fudd0o/+KLpX/7N7ufX1Ii3Xyz6UQAEBthF/zetLS0KCMjo/t9VlaWmpubNWXK\nlO5t8+fP1/z58/tymJgbNkx6/nnp7/5Oam2VBgwwnQgAnNengu+kcxfzDwaDCgaDcT3+zJnSv/yL\n9OST0ve/H9dDA0BYGhoa+vSgqIhm6YRCIU2dOlU7d+6UZLd0gsGgWltbJUnz5s1TaWlpjxF+WCEM\nzdL5vIMH7dbOm29K111nOg0AXFhc77RNSkqSZM/UCYVCqq+vV35+fl92aVRamvSDH0h///f2YmsA\nkEjCLvhlZWUqKCjQnj17lJaWptWrV0uSli1bpqqqKhUXF6u6ulopKSlRBXHymbZ9UV0tnTwprVpl\nOgkA9I5n2jroz3+Wiovtn8OGmU4DAL1z7Z22FwzhsoIv2evtHDgg/eIXppMAQO88u1qmW1o6XX7w\nA3sJ5Y0bTScBgJ5o6cTAli1SZaXU1iYNGmQ6DQD0REvHYffcIw0eLD3zjOkkANATLR2HPf20tHat\n3d4BADegpRNDa9bYI/zt26V+/UynAQCbZ0f4bvbtb0spKdKyZaaTAED0GOGHad8+KT/fHuVffbXp\nNADg4RG+W3v4XcaOlR5+WLrvPsnl5yYACY4efhycPi3l5UmLFtlPyQIAk5iWGWMtLdLUqfbc/MGD\nTacB4GcU/Dh48EGpo0P6//XjAMAIevhx8MMfSq+/br8AIN7o4cfZq69KDz1kr6h52WWm0wDwI8+O\n8L1m6lQpN9ce7QOAFzDC74PDh6Ubb5T+8Af7JwDEEyP8OBo+XHr8cfuRiGfPmk4DABfmmoLvpYu2\n57r3Xnt9nRUrTCcB4BdctDVo1y6psFBqbbUfhA4A8UBLx4DMTOmBB+wHoH/2mek0ANA7RvgOOXVK\nuvVWadw46ac/lQIB04kAJDpG+IYMGGA//3b7dukf/oEF1gC4DwXfQUlJ0qZN0muvMT8fgPtcYjpA\nohk8WKqvlyZOlC6/XPrud00nAgCbawp+TU2NgsGggsGg6Sh9NmyYtGWLVFQkDRxor6EPAE5paGiI\naho7F21jaN8+KRi0b866+27TaQAkmkhrp2tG+Ilo7Fjp97+XbrnFXmBt1izTiQD4GQU/xjIz7Qu5\nJSV2e2fKFNOJAPgVs3TiYNw4acMG6TvfsRdaAwATKPhxkp8v/eY30l13SW+9ZToNAD+i4MfRxInS\nv/6r9I1vSH/8o+k0APyGgh9nt98u/dM/2b38//xP02kA+AkXbQ2YMUM6ccIu/m+8IV17relEAPyA\ngm9IWZn0ySfSbbdJjY3SmDGmEwFIdK4p+Il0p224Kiuljz+WioulpiZpxAjTiQB4AXfaetgTT0hr\n1tgj/SFDTKcB4BXcaetBjzxij/RLSqTXX5eSk00nApCImKXjEj/8ob3uTnGx9O67ptMASEQUfJcI\nBKRnnpGmTZMKCqT/+i/TiQAkGlo6LhIISIsXSyNH2jdprV8vff3rplMBSBSM8F2oslJatcoe7f/u\nd6bTAEgUjPBdavJk+1GJ06ZJhw5J1dWmEwHwOqZlutz+/VJpqTRzpvSjH9ltHwCQIq+dFHwPOHJE\nuuMOKT1dWrlS6t/fdCIAbhBp7aSH7wEpKfb8/OPH7UXXjh83nQiAF8W04O/evVv33XefKisrtW7d\nulgeKuENHCj99rfSNddIhYV2Xx8AIhGXls6nn36qe+65Ry+//HLvIWjphM2ypB//WHrhBfuiblaW\n6UQATIlJS6eiokKpqanKzs7usb2pqUmZmZlKT09XbW1tr7+7YcMGTZo0SXfeeWfYofDFAgF7KYbH\nHpMmTZK2bjWdCIBXhDXC37p1qwYNGqS7775bO3fu7N6em5ur5557TqNHj9btt9+uN998U6+99pp2\n7NihRYsWacQ5yz9OmzZNGzZs6D0EI/yo1NdL3/qW9LOfSX/zN6bTAIi3mCyeVlhYqFAo1GNbR0eH\nJKmoqEiSVFJSom3btqm8vFzl5eWSpMbGRq1bt06WZWnWrFlhh0J4brtN+v3v7Rk8//u/0oMPmk4E\nwM2ivvGqpaVFGRkZ3e+zsrLU3NysKVOmdG+bOHGiJk6c2LeEuKCcHPuh6KWl0sGD0lNPSRcx9wpA\nL1xzp21NTU33f/vtQSh9NXq0XfSnT7efpPXSS9Kll5pOBcBp0T74pEvYs3RCoZCmTp3a3cPv6OhQ\nMBhUa2urJGnevHkqLS3tMcIPOwQ9fEecPCl9+9vS0aPSK69IV1xhOhGAWIrbjVdJSUmS7Jk6oVBI\n9fX1ys/Pj3Z3cMCll0q//KWUmWmvrf/ee6YTAXCTsAp+WVmZCgoKtGfPHqWlpWn16tWSpGXLlqmq\nqkrFxcWqrq5WSkpK1EFqamr69KcKbBdfLC1fbrd3br5Z2rfPdCIATmtoaOjRBg8Xa+kksBdekB59\nVNq4UcrNNZ0GgNM8+0zbmpoaLtY6bO5cex2e22+XfvUru80DwPuivXjLCN8H3nhDmj1bWrHCXmYZ\nQGLw7AgfsTNpkrR5s73S5pEjUlWV6UQATKDg+0RurtTUZLd33ntP+v73eZgK4DeuuSeTWTqxd801\n9g1a69dL8+ZJZ8+aTgQgGszSQdg6OqQZM6QhQ6Q1a6QBA0wnAhANnniFL5WUZK+lf/as3dfv7DSd\nCEA8uKbg09KJr0svtadqXnONPV3z/fdNJwIQLlo6iIplSTU10tq19lLLV11lOhGAcDEtExEJBKQl\nS6ShQ+1n5W7cKI0bZzoVgFig4EOSdP/99kXc226Tfv1riccYAImHlg56+MMf7LtyR42SxoyxWzxj\nxvR8feUrRiMC+H+ebemwlo473HqrvcLmf/+3dOCAFApJu3fbs3pCIfs1cOBfin9vJ4TLLzcWH/AF\n1tJBXFiW1N7+l5NB16vr/f/8j/0XwJgx0tix9iygc38OG8YdvoBTIq2dFHw46rPP7Cme+/fbfyl0\nvfbutX9+/LFd+LteXSeCsWPtNtIlYf7NeeaM9MEH0rFjvb+OHrV/9usnXXmlNHiw/bO31xVX8Bxg\neBMFH652/HjvJ4J9++w1ftLS/nIiSErqWdS7ivixY/aJ46tfvXAxT062TwxfdlI4d1+97SPckxAQ\nibQ06Tvf6ds+KPjwrFOn7NZQ1wmgo+OLi7mTo/LTp6UPP+z9pPDBB6w5hNgYPVq6996+7YOLtvCs\nAQOkjAz7FU/9+tlTUocMie9xgWhx0RYAfIbF0wAAvaLgA4BPUPABwCco+ADgExR8APAJ1xR8HoAC\nAOHhASgA4DNMywQA9IqCDwA+QcEHAJ+g4AOAT1DwAcAnKPgA4BMUfADwCQo+APiEawo+d9oCQHi4\n0xYAfIY7bQEAvaLgA4BPUPABwCco+ADgExR8APAJCj4A+AQFHwB8goIPAD5BwQcAn4h5wf/444+V\nl5enjRs3xvpQAIALiHnBf/LJJzV79uxYHwYA8CXCKvgVFRVKTU1VdnZ2j+1NTU3KzMxUenq6amtr\nz/u9+vp6ZWVlaciQIc6kRVhYhM5ZfJ7O4bM0K6yCP2fOHG3atOm87QsWLFBdXZ22bNmi5cuX68iR\nI1qzZo0WLlyoQ4cOqbGxUc3NzVq7dq1+/vOfs0BanPA/lbP4PJ3DZ2nWJeH8o8LCQoVCoR7bOjo6\nJElFRUWSpJKSEm3btk3l5eUqLy+XJD322GOSpJdeeklDhgxRIBBwKjcAIEJR9/BbWlqUkZHR/T4r\nK0vNzc29/tt77rlHkydPjvZQAAAHhDXCjwdG/85asmSJ6QgJhc/TOXyW5kRd8PPy8rRo0aLu921t\nbSotLY1qX/T2ASD2om7pJCUlSbJn6oRCIdXX1ys/P9+xYAAAZ4VV8MvKylRQUKA9e/YoLS1Nq1ev\nliQtW7ZMVVVVKi4uVnV1tVJSUiI6+JdN60RkxowZoxtvvFG5ubkaP3686Tie09v0487OTk2fPl2j\nRo3SjBkz9NFHHxlM6B29fZY1NTUaOXKkcnNzlZub2+vMP/Tu4MGDmjRpkq6//noFg0GtXbtWUhTf\nT8ugnJwcq7Gx0QqFQtZ1111ntbe3m4zjeWPGjLGOHj1qOoZnNTU1WTt27LBuuOGG7m1Lly61Hnjg\nAevkyZPW/fffbz311FMGE3pHb59lTU2N9fTTTxtM5V2HDx+2WltbLcuyrPb2duuqq66yjh8/HvH3\n09haOudO6xw9enT3tE70jcX1kKgVFhYqOTm5x7bt27ersrJSAwYMUEVFBd/RMPX2WUp8P6M1bNgw\n5eTkSJJSUlJ0/fXXq6WlJeLvp7GCH8m0ToQnEAjolltu0YwZM7RhwwbTcRLCud/TjIwMbd++3XAi\nb6utrdWECRO0dOlSdXZ2mo7jSXv37lVbW5vGjx8f8feT1TITyFtvvaV33nlHTzzxhB566CG9++67\npiN5HiNS59x33306cOCANm/erH379qmurs50JM/p7OzU7Nmz9eyzz2rQoEERfz+NFfy8vDzt3r27\n+31bW5smTJhgKk5CGD58uCQpMzNT06ZN06uvvmo4kffl5eVp165dkqRdu3YpLy/PcCLvGjp0qAKB\ngJKSknT//fdr/fr1piN5yunTpzVz5kyVl5dr+vTpkiL/fhor+EzrdNYnn3zS/Sdye3u7Nm/eHPV9\nEfiL/Px8rVq1SidOnNCqVasYlPTB4cOHJUlnzpzR2rVrufs+ApZlqbKyUjfccIMefPDB7u0Rfz9j\nfHH5ghoaGqyMjAxr7Nix1nPPPWcyiuft37/fGjdunDVu3DjrlltusVauXGk6kufcdddd1vDhw63+\n/ftbI0eOtFatWmUdP37cmjZtmpWWlmZNnz7d6uzsNB3TE7o+y379+lkjR460Vq5caZWXl1vZ2dnW\n1772NWvhwoXMKIvA1q1brUAgYI0bN87KycmxcnJyrNdeey3i72fAsmhSAoAfcNEWAHyCgg8APkHB\nBwCfoOADgE9Q8AHAJyj4AOATFHwA8In/A4iEz18jsp3BAAAAAElFTkSuQmCC\n"
      }
     ], 
     "prompt_number": 246
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "simseq = {}", 
      "for i in range(len(csfs)):", 
      "    simseq[i] = {}", 
      "for sn,ss in seqd.items():", 
      "    for i in range(len(csfs)):", 
      "        simseq[i][sn] = []", 
      "    for c,nvar in zip(ss,[[i for i in range(len(csfs)) if n<csfs[i]][0] for n in numpy.random.random_sample(size=len(ss))]):", 
      "        if nvar == 0:", 
      "            for i in range(len(csfs)):", 
      "                simseq[i][sn].append(c)", 
      "        else:", 
      "            alt = numpy.random.permutation(list(set('ACGT')-set(c)))[0]", 
      "            this_ord = numpy.random.permutation(len(csfs))", 
      "            for i in this_ord[:nvar]:", 
      "                simseq[i][sn].append(alt)", 
      "            for i in this_ord[nvar:]:", 
      "                simseq[i][sn].append(c)", 
      "", 
      "for i in range(len(csfs)):", 
      "    for sn in simseq[i].keys():", 
      "        simseq[i][sn] = ''.join(simseq[i][sn])"
     ], 
     "language": "python", 
     "outputs": [], 
     "prompt_number": 247
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "try:", 
      "    os.makedirs(outdir)", 
      "except:", 
      "    pass", 
      "for ind,simseqd in simseq.items():", 
      "    outf = open(outdir+'hap%s_agouti-mc1r.fa' % ind,'w')", 
      "    for sn,ss in simseqd.items():", 
      "        outf.write('>%s\\n%s\\n' % (sn,ss))", 
      "    outf.close()"
     ], 
     "language": "python", 
     "outputs": [], 
     "prompt_number": 248
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "num_ind = num_haplo/2", 
      "tot_seq = sum(map(len,seqd.values()))", 
      "# number of bases generated per haplo / bp per read-pair", 
      "nreads = int((tot_seq*target_cov/2.0) / 200)", 
      "print 'target coverage %s; reads per haplotype %s' % (target_cov,nreads)", 
      "try:", 
      "    os.makedirs(outdir+'sim-reads/')", 
      "except:", 
      "    pass", 
      "ind_haplos = reduce(lambda x,y:x+y, [[outdir+'sim-reads/sim%sa' % i,outdir+'sim-reads/sim%sb'% i] for i in range(1,num_ind+1)])", 
      "cmds = []", 
      "map_reference = os.path.expanduser('~/code/AFS/agouti-mc1r.masked.fasta')", 
      "for read_prefix, source_reference in zip(ind_haplos,numpy.random.permutation(glob(outdir+'hap*_agouti-mc1r.fa'))):", 
      "    cmds.append('simulate_and_map.py %s %s %s %s %s %s.vcf' % (read_prefix,os.path.basename(read_prefix)[:-1],source_reference,map_reference,nreads,read_prefix))", 
      "len(cmds)"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "target coverage 8; reads per haplotype 6913"
       ]
      }, 
      {
       "output_type": "pyout", 
       "prompt_number": 249, 
       "text": [
        "20"
       ]
      }
     ], 
     "prompt_number": 249
    }, 
    {
     "cell_type": "code", 
     "collapsed": true, 
     "input": [
      "for cmd in cmds:", 
      "    !$cmd"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:18 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap4_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap4_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:18 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:18 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:19 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:19 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:21 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.11 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.204 sec; CPU: 0.168 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_1_sequence.txt", 
        "[main] Real time: 0.437 sec; CPU: 0.378 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_2_sequence.txt", 
        "[main] Real time: 0.407 sec; CPU: 0.397 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[infer_isize] (25, 50, 75) percentile: (188, 200, 213)", 
        "[infer_isize] low and high boundaries: 138 and 263 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2526 pairs: 199.932 +/- 19.326", 
        "[infer_isize] skewness: -0.037; kurtosis: -0.006; ap_prior: 1.98e-05", 
        "[infer_isize] inferred maximum insert size: 309 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.03 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 26 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 919 out of 1294 Q17 singletons are mated.", 
        "[bwa_paired_sw] 3 out of 4 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a_s_1_2_sequence.txt", 
        "[main] Real time: 0.813 sec; CPU: 0.507 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:24 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a RGPL=illumina RGPU=lane1 RGSM=sim1    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:24 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:12:24\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a SM=sim1", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:26 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.04 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:29 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap3_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap3_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:29 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:30 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:31 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:31 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:33 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_2_sequence.txt created"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.11 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.182 sec; CPU: 0.169 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_1_sequence.txt", 
        "[main] Real time: 0.397 sec; CPU: 0.388 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_2_sequence.txt", 
        "[main] Real time: 0.463 sec; CPU: 0.406 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[infer_isize] (25, 50, 75) percentile: (186, 200, 212)", 
        "[infer_isize] low and high boundaries: 134 and 264 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2581 pairs: 199.114 +/- 19.399", 
        "[infer_isize] skewness: -0.020; kurtosis: 0.046; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 309 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 32 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 846 out of 1245 Q17 singletons are mated.", 
        "[bwa_paired_sw] 1 out of 1 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.29 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        " sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b_s_1_2_sequence.txt", 
        "[main] Real time: 0.919 sec; CPU: 0.501 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:36 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b RGPL=illumina RGPU=lane1 RGSM=sim1    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:36 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)", 
        "INFO\t"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "2012-07-24 04:12:36\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b SM=sim1", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:37 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:40 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap2_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap2_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:40 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:40 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:42 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:42 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:43 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... 0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.160 sec; CPU: 0.157 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_1_sequence.txt", 
        "[main] Real time: 0.402 sec; CPU: 0.394 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_2_sequence.txt", 
        "[main] Real time: 0.477 sec; CPU: 0.419 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (186, 200, 212)", 
        "[infer_isize] low and high boundaries: 134 and 264 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2686 pairs: 199.341 +/- 19.434", 
        "[infer_isize] skewness: 0.017; kurtosis: 0.085; ap_prior: 1.12e-05", 
        "[infer_isize] inferred maximum insert size: 309 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 37 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 852 out of 1269 Q17 singletons are mated.", 
        "[bwa_paired_sw] 1 out of 2 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a_s_1_2_sequence.txt", 
        "[main] Real time: 0.814 sec; CPU: 0.509 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:46 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a RGPL=illumina RGPU=lane1 RGSM=sim2    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:46 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:12:46\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a SM=sim2", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:47 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.02 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:50 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap19_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap19_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:50 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:50 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:52 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:52 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:53 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.169 sec; CPU: 0.160 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_1_sequence.txt", 
        "[main] Real time: 0.389 sec; CPU: 0.380 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_2_sequence.txt", 
        "[main] Real time: 0.448 sec; CPU: 0.415 sec", 
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[infer_isize] (25, 50, 75) percentile: (187, 200, 212)", 
        "[infer_isize] low and high boundaries: 137 and 262 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2554 pairs: 198.878 +/- 19.193", 
        "[infer_isize] skewness: -0.067; kurtosis: 0.151; ap_prior: 1.57e-05", 
        "[infer_isize] inferred maximum insert size: 307 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 31 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 882 out of 1275 Q17 singletons are mated.", 
        "[bwa_paired_sw] 2 out of 4 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b_s_1_2_sequence.txt", 
        "[main] Real time: 0.850 sec; CPU: 0.524 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:56 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b RGPL=illumina RGPU=lane1 RGSM=sim2    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:12:56 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:12:56\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b SM=sim2", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:12:58 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:01 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap17_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap17_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:01 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:01 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:02 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:02 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:04 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.09 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... 0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.159 sec; CPU: 0.156 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_1_sequence.txt", 
        "[main] Real time: 0.401 sec; CPU: 0.385 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_2_sequence.txt", 
        "[main] Real time: 0.450 sec; CPU: 0.401 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (187, 200, 214)", 
        "[infer_isize] low and high boundaries: 133 and 268 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2614 pairs: 199.821 +/- 20.074", 
        "[infer_isize] skewness: -0.004; kurtosis: -0.049; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 313 (5.63 sigma)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 34 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 896 out of 1325 Q17 singletons are mated.", 
        "[bwa_paired_sw] 0 out of 1 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.31 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a_s_1_2_sequence.txt", 
        "[main] Real time: 0.840 sec; CPU: 0.518 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:06 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a RGPL=illumina RGPU=lane1 RGSM=sim3    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:06 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)", 
        "INFO\t"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "2012-07-24 04:13:06\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a SM=sim3", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:08 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.02 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:10 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap7_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap7_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:10 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:10 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:12 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:12 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:14 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.165 sec; CPU: 0.160 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_1_sequence.txt", 
        "[main] Real time: 0.397 sec; CPU: 0.392 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_2_sequence.txt", 
        "[main] Real time: 0.418 sec; CPU: 0.396 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (186, 200, 212)", 
        "[infer_isize] low and high boundaries: 134 and 264 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2578 pairs: 199.278 +/- 19.291", 
        "[infer_isize] skewness: 0.012; kurtosis: -0.107; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 21 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 887 out of 1294 Q17 singletons are mated.", 
        "[bwa_paired_sw] 1 out of 1 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b_s_1_2_sequence.txt", 
        "[main] Real time: 0.828 sec; CPU: 0.514 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:16 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b RGPL=illumina RGPU=lane1 RGSM=sim3    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:16 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:13:17\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b SM=sim3", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:18 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:21 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap16_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap16_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:21 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:21 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:22 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:22 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:24 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.179 sec; CPU: 0.162 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_1_sequence.txt", 
        "[main] Real time: 0.407 sec; CPU: 0.383 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_2_sequence.txt", 
        "[main] Real time: 0.430 sec; CPU: 0.406 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[infer_isize] (25, 50, 75) percentile: (186, 199, 211)", 
        "[infer_isize] low and high boundaries: 136 and 261 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2586 pairs: 198.655 +/- 19.453", 
        "[infer_isize] skewness: 0.068; kurtosis: -0.056; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 26 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 813 out of 1255 Q17 singletons are mated.", 
        "[bwa_paired_sw] 0 out of 0 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.29 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a_s_1_2_sequence.txt", 
        "[main] Real time: 0.884 sec; CPU: 0.507 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:27 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a RGPL=illumina RGPU=lane1 RGSM=sim4    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:27 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:13:27\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a SM=sim4", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:28 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.02 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:31 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap0_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap0_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:31 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:31 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:32 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:32 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:34 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.159 sec; CPU: 0.157 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_1_sequence.txt", 
        "[main] Real time: 0.394 sec; CPU: 0.388 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_2_sequence.txt", 
        "[main] Real time: 0.444 sec; CPU: 0.407 sec", 
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[infer_isize] (25, 50, 75) percentile: (186, 200, 212)", 
        "[infer_isize] low and high boundaries: 134 and 264 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2521 pairs: 199.230 +/- 19.280", 
        "[infer_isize] skewness: -0.007; kurtosis: 0.041; ap_prior: 1.19e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 30 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 878 out of 1284 Q17 singletons are mated.", 
        "[bwa_paired_sw] 1 out of 3 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b_s_1_2_sequence.txt", 
        "[main] Real time: 0.800 sec; CPU: 0.509 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:37 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b RGPL=illumina RGPU=lane1 RGSM=sim4    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:37 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)", 
        "INFO\t2012-07-24 04:13:37\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b SM=sim4", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:38 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.02 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:41 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap10_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap10_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:41 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:41 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:42 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:42 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:44 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.11 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.185 sec; CPU: 0.173 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_1_sequence.txt", 
        "[main] Real time: 0.433 sec; CPU: 0.400 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_2_sequence.txt", 
        "[main] Real time: 0.447 sec; CPU: 0.398 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (186, 200, 212)", 
        "[infer_isize] low and high boundaries: 134 and 264 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2566 pairs: 198.973 +/- 19.349", 
        "[infer_isize] skewness: 0.012; kurtosis: 0.052; ap_prior: 1.95e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 27 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 839 out of 1242 Q17 singletons are mated.", 
        "[bwa_paired_sw] 2 out of 5 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.29 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a_s_1_2_sequence.txt", 
        "[main] Real time: 0.801 sec; CPU: 0.511 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:47 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a RGPL=illumina RGPU=lane1 RGSM=sim5    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:47 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:13:47\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a SM=sim5", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:48 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.02 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:51 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap14_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap14_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:51 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:51 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:53 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:53 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:55 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.11 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.181 sec; CPU: 0.171 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_1_sequence.txt", 
        "[main] Real time: 0.416 sec; CPU: 0.395 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_2_sequence.txt", 
        "[main] Real time: 0.423 sec; CPU: 0.406 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (187, 200, 212)", 
        "[infer_isize] low and high boundaries: 137 and 262 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2568 pairs: 199.336 +/- 19.262", 
        "[infer_isize] skewness: 0.017; kurtosis: 0.034; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] time elapses: 0.03 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 36 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 868 out of 1277 Q17 singletons are mated.", 
        "[bwa_paired_sw] 1 out of 2 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b_s_1_2_sequence.txt", 
        "[main] Real time: 0.828 sec; CPU: 0.502 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:57 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b RGPL=illumina RGPU=lane1 RGSM=sim5    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:13:57 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:13:58\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b SM=sim5", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:13:59 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.02 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:01 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap6_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap6_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:01 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:01 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:03 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:03 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:05 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_2_sequence.txt created"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.167 sec; CPU: 0.162 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_1_sequence.txt", 
        "[main] Real time: 0.407 sec; CPU: 0.391 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_2_sequence.txt", 
        "[main] Real time: 0.427 sec; CPU: 0.399 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (186, 199, 211)", 
        "[infer_isize] low and high boundaries: 136 and 261 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2583 pairs: 198.663 +/- 19.373", 
        "[infer_isize] skewness: -0.021; kurtosis: -0.086; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 23 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 898 out of 1321 Q17 singletons are mated.", 
        "[bwa_paired_sw] 1 out of 2 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.31 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a_s_1_2_sequence.txt", 
        "[main] Real time: 0.800 sec; CPU: 0.514 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:07 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a RGPL=illumina RGPU=lane1 RGSM=sim6    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:07 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:14:07\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a SM=sim6", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:09 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.02 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:11 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap11_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap11_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:11 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:11 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:13 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:13 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:14 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.171 sec; CPU: 0.162 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_1_sequence.txt", 
        "[main] Real time: 0.388 sec; CPU: 0.384 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_2_sequence.txt", 
        "[main] Real time: 0.429 sec; CPU: 0.401 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (186, 200, 212)", 
        "[infer_isize] low and high boundaries: 134 and 264 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2540 pairs: 199.299 +/- 19.629", 
        "[infer_isize] skewness: 0.075; kurtosis: -0.008; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 310 (5.64 sigma)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 38 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 857 out of 1279 Q17 singletons are mated.", 
        "[bwa_paired_sw] 1 out of 1 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b_s_1_2_sequence.txt", 
        "[main] Real time: 0.793 sec; CPU: 0.505 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:17 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b RGPL=illumina RGPU=lane1 RGSM=sim6    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:17 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:14:17\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b SM=sim6", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:19 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:22 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap8_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap8_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:22 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:22 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:24 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:24 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:26 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.177 sec; CPU: 0.162 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_1_sequence.txt", 
        "[main] Real time: 0.415 sec; CPU: 0.393 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_2_sequence.txt", 
        "[main] Real time: 0.426 sec; CPU: 0.405 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (186, 200, 213)", 
        "[infer_isize] low and high boundaries: 132 and 267 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2545 pairs: 199.497 +/- 19.299", 
        "[infer_isize] skewness: -0.105; kurtosis: -0.071; ap_prior: 1.18e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 33 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 911 out of 1341 Q17 singletons are mated.", 
        "[bwa_paired_sw] 0 out of 1 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.31 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a_s_1_2_sequence.txt", 
        "[main] Real time: 0.867 sec; CPU: 0.524 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:29 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a RGPL=illumina RGPU=lane1 RGSM=sim7    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:29 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:14:29\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a SM=sim7", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:31 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:33 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap5_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap5_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:33 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:33 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:35 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:35 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:36 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... 0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.166 sec; CPU: 0.160 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_1_sequence.txt", 
        "[main] Real time: 0.411 sec; CPU: 0.394 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.32 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_2_sequence.txt", 
        "[main] Real time: 0.523 sec; CPU: 0.412 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[infer_isize] (25, 50, 75) percentile: (186, 200, 212)", 
        "[infer_isize] low and high boundaries: 134 and 264 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2543 pairs: 199.229 +/- 19.278", 
        "[infer_isize] skewness: 0.039; kurtosis: -0.021; ap_prior: 3.14e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 29 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 867 out of 1285 Q17 singletons are mated.", 
        "[bwa_paired_sw] 4 out of 8 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b_s_1_2_sequence.txt", 
        "[main] Real time: 0.848 sec; CPU: 0.519 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:39 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b RGPL=illumina RGPU=lane1 RGSM=sim7    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:39 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:14:39\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b SM=sim7", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:42 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.04 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:45 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap1_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap1_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:45 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:45 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:46 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:46 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:48 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... 0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.179 sec; CPU: 0.164 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_1_sequence.txt", 
        "[main] Real time: 0.397 sec; CPU: 0.385 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.32 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_2_sequence.txt", 
        "[main] Real time: 0.556 sec; CPU: 0.421 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (187, 199, 211)", 
        "[infer_isize] low and high boundaries: 139 and 259 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2572 pairs: 199.191 +/- 19.148", 
        "[infer_isize] skewness: -0.007; kurtosis: -0.035; ap_prior: 1.17e-05", 
        "[infer_isize] inferred maximum insert size: 307 (5.64 sigma)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 22 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 874 out of 1274 Q17 singletons are mated.", 
        "[bwa_paired_sw] 1 out of 3 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a_s_1_2_sequence.txt", 
        "[main] Real time: 0.843 sec; CPU: 0.508 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:51 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a RGPL=illumina RGPU=lane1 RGSM=sim8    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:51 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)", 
        "INFO\t"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "2012-07-24 04:14:51\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a SM=sim8", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:52 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:55 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap12_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap12_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:55 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:55 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:56 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:14:56 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:14:58 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_2_sequence.txt created"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.11 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.270 sec; CPU: 0.176 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_1_sequence.txt", 
        "[main] Real time: 0.858 sec; CPU: 0.390 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_2_sequence.txt", 
        "[main] Real time: 0.741 sec; CPU: 0.400 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[infer_isize] (25, 50, 75) percentile: (186, 200, 213)", 
        "[infer_isize] low and high boundaries: 132 and 267 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2550 pairs: 199.053 +/- 19.958", 
        "[infer_isize] skewness: -0.023; kurtosis: -0.041; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 312 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 31 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 883 out of 1259 Q17 singletons are mated.", 
        "[bwa_paired_sw] 0 out of 2 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b_s_1_2_sequence.txt", 
        "[main] Real time: 2.055 sec; CPU: 0.520 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:05 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b RGPL=illumina RGPU=lane1 RGSM=sim8    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:05 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:15:05\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b SM=sim8", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:07 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:12 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap15_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap15_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:12 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:12 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:13 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:13 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:15 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_2_sequence.txt created"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... 0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.210 sec; CPU: 0.166 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_1_sequence.txt", 
        "[main] Real time: 0.393 sec; CPU: 0.382 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_2_sequence.txt", 
        "[main] Real time: 0.415 sec; CPU: 0.405 sec", 
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[infer_isize] (25, 50, 75) percentile: (186, 199, 213)", 
        "[infer_isize] low and high boundaries: 132 and 267 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2568 pairs: 199.018 +/- 19.762", 
        "[infer_isize] skewness: -0.022; kurtosis: -0.125; ap_prior: 2.33e-05", 
        "[infer_isize] inferred maximum insert size: 310 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 29 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 872 out of 1288 Q17 singletons are mated.", 
        "[bwa_paired_sw] 2 out of 6 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.31 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a_s_1_2_sequence.txt", 
        "[main] Real time: 0.993 sec; CPU: 0.518 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:18 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a RGPL=illumina RGPU=lane1 RGSM=sim9    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:18 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:15:18\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a SM=sim9", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:21 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.05 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:25 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap18_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap18_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:25 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:25 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:26 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:26 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:28 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_2_sequence.txt created", 
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.175 sec; CPU: 0.161 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_1_sequence.txt", 
        "[main] Real time: 0.392 sec; CPU: 0.385 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.30 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_2_sequence.txt", 
        "[main] Real time: 0.396 sec; CPU: 0.392 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (186, 199, 212)", 
        "[infer_isize] low and high boundaries: 134 and 264 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2639 pairs: 198.931 +/- 19.569", 
        "[infer_isize] skewness: 0.011; kurtosis: 0.001; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 309 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 36 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 859 out of 1310 Q17 singletons are mated.", 
        "[bwa_paired_sw] 0 out of 2 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.30 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b_s_1_2_sequence.txt", 
        "[main] Real time: 0.794 sec; CPU: 0.517 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:31 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b RGPL=illumina RGPU=lane1 RGSM=sim9    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:31 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:15:31\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b SM=sim9", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:32 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.02 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:36 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap13_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap13_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:36 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:36 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:38 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:38 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:40 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.04 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_2_sequence.txt created"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.11 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.247 sec; CPU: 0.174 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_1_sequence.txt", 
        "[main] Real time: 0.735 sec; CPU: 0.418 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.32 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_2_sequence.txt", 
        "[main] Real time: 0.654 sec; CPU: 0.419 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[infer_isize] (25, 50, 75) percentile: (187, 199, 212)", 
        "[infer_isize] low and high boundaries: 137 and 262 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2552 pairs: 199.391 +/- 19.294", 
        "[infer_isize] skewness: 0.027; kurtosis: 0.003; ap_prior: 1.18e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)", 
        "[bwa_sai2sam_pe_core] time elapses: 0.04 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 28 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 845 out of 1262 Q17 singletons are mated.", 
        "[bwa_paired_sw] 1 out of 3 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.29 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.05 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a_s_1_2_sequence.txt", 
        "[main] Real time: 1.221 sec; CPU: 0.524 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:45 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a RGPL=illumina RGPU=lane1 RGSM=sim10    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:45 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "2012-07-24 04:15:45\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a SM=sim10", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:48 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.05 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:52 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary REFERENCE=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap9_agouti-mc1r.fa OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/hap9_agouti-mc1r.dict    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:52 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:52 EDT 2012] net.sf.picard.sam.CreateSequenceDictionary done. Elapsed time: 0.00 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:53 EDT 2012] net.sf.picard.sam.SamToFastq INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b-simreads.sorted.bam FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_1_sequence.txt SECOND_END_FASTQ=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_2_sequence.txt INCLUDE_NON_PF_READS=true VALIDATION_STRINGENCY=SILENT    OUTPUT_PER_RG=false RE_REVERSE=true READ1_TRIM=0 READ2_TRIM=0 INCLUDE_NON_PRIMARY_ALIGNMENTS=false VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:53 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:55 EDT 2012] net.sf.picard.sam.SamToFastq done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192", 
        "", 
        "/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_2_sequence.txt created"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] Pack FASTA... 0.01 sec", 
        "[bwa_index] Construct BWT for the packed sequence..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_index] 0.10 seconds elapse.", 
        "[bwa_index] Update BWT... 0.00 sec", 
        "[bwa_index] Pack forward-only FASTA... 0.01 sec", 
        "[bwa_index] Construct SA from BWT and Occ... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa index /Users/brant/code/AFS/agouti-mc1r.masked.fasta", 
        "[main] Real time: 0.179 sec; CPU: 0.167 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.29 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_1_sequence.txt", 
        "[main] Real time: 0.408 sec; CPU: 0.392 sec", 
        "[bwa_aln] 17bp reads: max_diff = 2", 
        "[bwa_aln] 38bp reads: max_diff = 3", 
        "[bwa_aln] 64bp reads: max_diff = 4", 
        "[bwa_aln] 93bp reads: max_diff = 5", 
        "[bwa_aln] 124bp reads: max_diff = 6", 
        "[bwa_aln] 157bp reads: max_diff = 7", 
        "[bwa_aln] 190bp reads: max_diff = 8", 
        "[bwa_aln] 225bp reads: max_diff = 9", 
        "[bwa_aln_core] calculate SA coordinate... "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.31 sec", 
        "[bwa_aln_core] write to the disk... 0.00 sec", 
        "[bwa_aln_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa aln /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_2_sequence.txt", 
        "[main] Real time: 0.411 sec; CPU: 0.408 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] convert to sequence coordinate... ", 
        "[infer_isize] (25, 50, 75) percentile: (187, 200, 212)", 
        "[infer_isize] low and high boundaries: 137 and 262 for estimating avg and std", 
        "[infer_isize] inferred external isize from 2576 pairs: 199.149 +/- 19.219", 
        "[infer_isize] skewness: -0.077; kurtosis: -0.085; ap_prior: 1.00e-05", 
        "[infer_isize] inferred maximum insert size: 308 (5.64 sigma)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_sai2sam_pe_core] time elapses: 0.03 sec", 
        "[bwa_sai2sam_pe_core] changing coordinates of 32 alignments.", 
        "[bwa_sai2sam_pe_core] align unmapped mate..."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[bwa_paired_sw] 911 out of 1308 Q17 singletons are mated.", 
        "[bwa_paired_sw] 0 out of 2 Q17 discordant pairs are fixed.", 
        "[bwa_sai2sam_pe_core] time elapses: 0.31 sec", 
        "[bwa_sai2sam_pe_core] refine gapped alignments... 0.01 sec", 
        "[bwa_sai2sam_pe_core] print alignments... [samopen] SAM header is present: 2 sequences."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "0.04 sec", 
        "[bwa_sai2sam_pe_core] 6913 sequences have been processed."
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[main] Version: 0.6.2-r126", 
        "[main] CMD: bwa sampe /Users/brant/code/AFS/agouti-mc1r.masked.fasta /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_1_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_2_sequence.txt.sai /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_1_sequence.txt /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b_s_1_2_sequence.txt", 
        "[main] Real time: 0.800 sec; CPU: 0.523 sec"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:15:58 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups INPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b-aln.sorted.bam OUTPUT=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b-aln.sorted.rg.bam RGID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b RGLB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b RGPL=illumina RGPU=lane1 RGSM=sim10    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false", 
        "[Tue Jul 24 04:15:58 EDT 2012] Executing as brant@gottuminum.local on Mac OS X 10.7.4 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.6.0_33-b03-424-11M3720; Picard version: 1.73(1234)"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO\t2012-07-24 04:15:58\tAddOrReplaceReadGroups\tCreated read group ID=/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b PL=illumina LB=lane1_/Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b SM=sim10", 
        ""
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "[Tue Jul 24 04:16:00 EDT 2012] net.sf.picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.03 minutes.", 
        "Runtime.totalMemory()=85000192"
       ]
      }
     ], 
     "prompt_number": 250
    }, 
    {
     "cell_type": "code", 
     "collapsed": true, 
     "input": [
      "bams = glob(outdir+'sim-reads/*.rg.bam')", 
      "Istr = ' -I '.join(bams)", 
      "vcf = outdir+'all.vcf'", 
      "java_ram = '1g'", 
      "jar = os.path.expanduser('~/jars/gatk/GenomeAnalysisTK.jar')", 
      "gatkcmd = 'java -Xmx%s -jar %s -T UnifiedGenotyper -I %s -R %s -o %s -maxAlleles 1 -out_mode EMIT_ALL_SITES' % (java_ram, jar, Istr, map_reference, vcf)", 
      "!$gatkcmd"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO  04:16:10,474 HelpFormatter - --------------------------------------------------------------------------------- ", 
        "INFO  04:16:10,477 HelpFormatter - The Genome Analysis Toolkit (GATK) v1.6-13-g91f02df, Compiled 2012/07/23 13:34:16 ", 
        "INFO  04:16:10,478 HelpFormatter - Copyright (c) 2010 The Broad Institute ", 
        "INFO  04:16:10,478 HelpFormatter - Please view our documentation at http://www.broadinstitute.org/gsa/wiki ", 
        "INFO  04:16:10,478 HelpFormatter - For support, please view our support site at http://getsatisfaction.com/gsa ", 
        "INFO  04:16:10,479 HelpFormatter - Program Args: -T UnifiedGenotyper -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim10b-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim1b-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim2b-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim3b-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim4b-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim5b-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim6b-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim7b-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim8b-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9a-aln.sorted.rg.bam -I /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/sim-reads/sim9b-aln.sorted.rg.bam -R /Users/brant/code/AFS/agouti-mc1r.masked.fasta -o /Users/brant/code/AFS/sim-20haplos_exp-dist_theta0.05_8xCov/all.vcf -maxAlleles 1 -out_mode EMIT_ALL_SITES ", 
        "INFO  04:16:10,481 HelpFormatter - Date/Time: 2012/07/24 04:16:10 ", 
        "INFO  04:16:10,481 HelpFormatter - --------------------------------------------------------------------------------- ", 
        "INFO  04:16:10,481 HelpFormatter - --------------------------------------------------------------------------------- "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO  04:16:10,526 GenomeAnalysisEngine - Strictness is SILENT "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO  04:16:10,599 SAMDataSource$SAMReaders - Initializing SAMRecords in serial "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO  04:16:10,735 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.13 "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO  04:16:11,338 TraversalEngine - [INITIALIZATION COMPLETE; TRAVERSAL STARTING] ", 
        "INFO  04:16:11,339 TraversalEngine -        Location processed.sites  runtime per.1M.sites completed total.runtime remaining "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:14,438 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:1421 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:14,920 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:1562 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:15,170 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:1683 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:16,125 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:2037 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:16,546 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:2660 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:17,053 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:2912 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:18,306 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:3528 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:18,598 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:4044 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:18,848 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:5097 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:18,975 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:5207 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:19,094 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:5356 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:19,427 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:5642 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:19,515 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:5734 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:19,520 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:5740 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:19,822 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:6380 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:20,146 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:6884 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:20,240 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:8467 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:20,356 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:8675 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:20,361 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:8686 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:20,410 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:8793 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:20,785 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:9951 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:20,840 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:10068 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:20,842 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:10072 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:20,928 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:10697 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:21,010 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:10797 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:21,233 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:11301 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:21,288 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:11413 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:21,464 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:11792 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:21,790 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:12071 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:21,903 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:12209 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:22,728 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:14583 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:22,863 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:15210 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:23,908 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:16825 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:24,121 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:17022 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:24,212 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:17223 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:25,143 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:19308 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:25,599 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:19711 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:25,777 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:20535 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:25,899 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:21393 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:26,093 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:22355 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:26,098 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:22374 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:26,224 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:22556 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:26,387 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:22613 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:26,397 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:22637 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:26,419 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:22653 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:26,734 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:24938 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:26,958 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:25920 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:26,982 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:25998 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,055 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:26264 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,071 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:26329 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:27,097 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:26428 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,151 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:26632 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,178 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:26779 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:27,197 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:26845 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,235 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:27239 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,465 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:28183 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:27,474 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:28223 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,529 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:28323 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,594 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:28559 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:27,621 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:28656 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:27,623 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:28663 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,716 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:29044 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:27,740 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:29141 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,749 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:29174 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:27,774 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:29282 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,820 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:29469 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:27,828 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:29508 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,861 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:29614 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:27,951 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:29932 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,045 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:30249 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,079 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:30367 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,128 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:30534 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,222 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:31258 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:28,249 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:31351 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,286 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:31596 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,412 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:32011 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,420 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:32041 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:28,452 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:32158 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,643 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:33071 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:28,673 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:33180 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,903 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:35987 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:28,904 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:35989 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,983 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:36277 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:28,998 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:36335 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:29,009 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:36382 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:29,024 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:36441 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:29,032 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:36461 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,053 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:36550 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:29,082 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:37003 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,139 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:37219 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:29,146 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:37248 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:29,152 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:37273 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,223 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:38566 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,314 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:38947 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,352 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:39111 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:29,360 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:39147 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,448 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:39471 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:29,456 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:39511 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,648 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:40390 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,825 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:41897 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,888 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:42262 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:29,899 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:42308 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:29,987 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:42672 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,200 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:43497 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,288 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:43752 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,325 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:46081 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:30,352 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:46177 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,438 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:46724 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:30,452 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:46768 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:30,468 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:46832 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,478 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:46869 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:30,488 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:46901 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,550 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:47143 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,652 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:47567 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:30,657 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:47588 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:30,663 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:47610 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:30,665 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:47621 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,732 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:48049 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,829 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:48419 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,855 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:48500 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,930 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:48793 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:30,932 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:48796 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:30,992 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:49043 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,250 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:50798 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:31,254 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:50812 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,271 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:50847 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:31,300 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:50901 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,339 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:51040 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,455 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:51453 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,481 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:51535 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,601 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:51796 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:31,619 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:51831 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:31,628 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:51835 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,642 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:51856 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,779 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:52541 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,808 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:52662 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:31,955 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:53215 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,002 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:53776 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,047 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:53879 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,143 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:54372 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,228 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:54741 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,281 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:55251 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:32,301 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:55349 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,305 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:55363 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:32,330 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:56361 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,457 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:59521 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,561 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:61801 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,591 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:61890 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,643 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:62051 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:32,647 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:62056 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:32,649 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:62061 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:32,664 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:62105 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,766 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:62784 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,806 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:62894 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,834 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:62976 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,936 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:63651 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:32,980 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:63826 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,072 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:64180 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,239 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:66017 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,281 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:66181 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,296 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:66252 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,377 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:66553 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,384 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:66583 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,388 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:66598 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,482 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:66908 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,555 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:67203 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,562 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:67232 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,626 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:67477 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,631 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:67492 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,655 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:67569 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,666 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:67593 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,668 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:67594 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,709 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:67757 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,712 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:67769 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,804 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:68287 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,813 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:68327 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:33,819 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:68348 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,933 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:68835 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:33,984 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:69063 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:34,014 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:69638 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:34,022 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:69663 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,111 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:70004 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,223 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:70759 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,320 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:71114 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:34,333 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:71172 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,351 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:71228 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:34,376 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:71322 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,399 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:72596 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:34,422 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:72681 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,487 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:73050 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,553 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:74100 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:34,554 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:74101 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:34,592 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:74249 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,599 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:74276 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,742 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:75297 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,811 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:76400 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:34,818 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:76430 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,891 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:76660 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:34,981 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:77030 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,028 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:77221 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,197 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:79003 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:35,212 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:79043 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,286 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:79227 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:35,327 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:79639 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:35,336 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:79669 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,369 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:79758 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:35,383 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:79819 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,504 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:80646 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:35,525 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:80716 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,560 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:80784 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,647 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:81392 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,671 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:81478 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:35,701 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:81580 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,785 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:82243 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,820 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:82367 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:35,909 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:82758 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,052 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:83310 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,107 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:83353 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,160 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:83492 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,238 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:86988 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,377 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:87297 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,404 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:87412 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:36,446 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:87556 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,573 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:87925 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,613 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:88202 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:36,628 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:88267 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,710 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:88601 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,766 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:89337 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:36,791 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:89447 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:36,805 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:89501 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,836 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:89610 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,886 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:89780 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:36,892 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:89801 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,938 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:90126 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:36,971 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:90264 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:37,013 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:90419 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,027 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:90464 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:37,043 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:90525 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:37,054 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:90938 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,122 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:91397 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,213 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:91742 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,227 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:91788 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:37,267 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:92457 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,296 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:92557 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,397 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:93256 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:37,426 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:93373 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,456 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:93651 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,529 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:93875 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,635 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:94181 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:37,900 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:100426 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:37,924 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:100528 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,045 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:101535 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,129 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:101929 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:38,149 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:101975 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,245 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:102289 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,335 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:102536 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,400 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:102792 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,413 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:102835 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:38,449 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:103062 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,501 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:103259 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,591 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:103574 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,704 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:104159 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:38,712 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:104195 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:38,715 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:104211 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,730 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:104260 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:38,740 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:104293 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:38,752 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:104329 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,803 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:104734 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,863 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:104901 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,888 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:104980 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:38,940 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:105564 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:39,007 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:105834 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:39,117 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:106289 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:39,121 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:106306 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:39,130 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:106354 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:39,202 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:107189 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:39,209 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:107219 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:39,241 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:107358 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:39,291 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:108784 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:39,396 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:109490 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:39,404 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:109521 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:39,409 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:109536 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:39,604 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:110767 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:39,644 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:110936 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:39,729 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:111281 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:39,744 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:111353 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:39,788 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:111525 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:39,932 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:113740 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:39,959 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:113852 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,033 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:115045 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:40,061 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:115153 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,084 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:115236 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:40,112 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:115332 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,140 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:115444 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,231 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:117299 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,272 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:117390 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:40,283 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:117425 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,334 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:117963 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,441 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:118387 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,521 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:118706 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,528 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:118732 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,652 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:120057 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:40,659 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:120083 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:40,673 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:120155 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,732 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:121320 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:40,761 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:121429 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,835 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:121631 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,876 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:121779 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:40,890 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:121852 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:40,893 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:121863 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:40,994 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:122296 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:41,014 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:122380 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:41,138 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:124064 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:41,208 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:124777 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO  04:16:41,258 TraversalEngine - agouti_bac:125001        1.25e+05   30.0 s        4.0 m     36.2%        83.1 s    53.0 s ", 
        "WARN  04:16:41,301 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:125313 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:41,326 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:125363 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:41,500 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:126624 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:41,534 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:126720 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:41,718 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:127791 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:41,735 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:127925 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:41,755 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:128007 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:41,760 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:128025 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:41,802 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:128170 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:41,918 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:128538 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,057 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:129559 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,081 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:129649 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:42,101 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:129723 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,326 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:131968 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,333 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:131998 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,400 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:132144 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,486 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:132463 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,642 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:133044 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,699 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:133275 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,895 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:134033 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:42,899 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:134049 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:42,904 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:134067 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:42,916 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:134201 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:42,929 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:134259 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:42,930 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:134260 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,036 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:134673 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,062 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:134728 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:43,082 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:134780 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,129 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:135172 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:43,131 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:135179 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:43,149 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:135240 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,195 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:135496 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,424 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:137838 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:43,437 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:137896 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,515 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:138147 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,623 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:138297 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,778 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:138540 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,828 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:138632 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:43,858 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:138667 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:43,869 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:138695 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,005 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:138875 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,177 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:139075 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,234 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:139091 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,370 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:139547 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,645 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:139912 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:44,660 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:139956 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,735 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:140229 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:44,756 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:140280 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,846 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:140823 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,895 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:140921 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:44,924 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:141184 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,952 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:141284 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:44,998 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:141406 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:45,004 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:141429 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:45,128 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:141699 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:45,318 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:142225 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:45,584 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:142817 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:45,836 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:143961 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:45,857 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:144038 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:45,898 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:144099 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:46,013 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:144628 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:46,160 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:145008 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:46,335 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:145335 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:46,370 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:145442 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:46,419 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:145564 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:46,496 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:145805 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:46,764 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:146376 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:46,795 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:146486 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:46,981 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:147274 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,024 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:147392 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:47,031 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:147400 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,170 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:147633 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:47,194 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:147728 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,210 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:147786 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,345 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:148596 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,371 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:148696 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:47,392 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:148769 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:47,410 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:148845 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,524 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:150176 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:47,555 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:150312 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,595 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:150474 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,634 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:150598 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:47,650 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:150655 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,817 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:152135 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,853 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:152251 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:47,858 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:152270 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,914 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:152686 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,945 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:152968 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:47,997 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:153186 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:48,139 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:154370 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:48,241 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:154787 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:48,248 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:155251 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:48,251 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:155264 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:48,268 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:155330 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:48,381 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:155795 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:48,390 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:155829 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:48,412 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:155902 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:48,638 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:158471 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,016 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:164408 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,113 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:164900 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,136 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:165323 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:49,138 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:165330 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:49,148 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:165381 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,184 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:165495 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:49,201 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:165557 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:49,202 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:165559 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,321 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:166691 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,367 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:166899 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,399 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:167045 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:49,414 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:167104 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,526 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:167552 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,582 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:167773 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,644 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:168819 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:49,660 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:168890 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,765 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:169373 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:49,964 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:171275 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,013 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:171488 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,096 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:174649 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,184 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:175951 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,219 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:176079 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,347 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:179476 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,361 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:179520 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:50,376 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:179547 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,414 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:179695 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:50,422 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:179709 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,596 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:181298 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,622 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:181403 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:50,630 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:181429 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:50,642 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:181481 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,711 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:182232 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:50,716 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:182249 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,808 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:182721 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,872 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:183135 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:50,914 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:183423 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,095 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:184733 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,160 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:184971 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,182 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:184981 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:51,209 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:185069 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,313 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:185277 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:51,320 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at agouti_bac:185304 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,407 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:331 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:51,419 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:379 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,463 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:544 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,509 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:849 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,650 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:1099 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,783 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:1804 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:51,978 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:2215 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:52,044 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:2294 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:52,071 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:2370 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:52,109 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:2442 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:52,126 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:2492 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:52,152 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:2555 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:52,254 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:3140 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:52,330 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:3297 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:52,353 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:3372 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:52,514 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:3701 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:52,738 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:5000 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:53,116 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:6625 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:53,213 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:6808 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:53,214 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:6811 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:53,278 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:6971 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:53,354 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:7057 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:53,361 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:7074 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:53,387 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:7158 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:53,388 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:7161 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:53,441 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:7247 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:53,671 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:8532 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:53,675 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:8547 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:53,868 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:9133 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,048 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:9620 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:54,073 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:9666 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:54,080 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:9689 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,195 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:9798 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,267 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:9880 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,331 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:10080 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,414 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:11149 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,492 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:11450 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:54,508 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:11510 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:54,518 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:11548 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,607 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:12240 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,648 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:12351 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,741 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:13047 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,817 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:13173 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,912 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:13402 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:54,926 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:13463 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:54,956 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:13847 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:55,065 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:14169 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:55,141 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:15189 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:55,299 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:15660 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:55,347 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:15833 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:55,576 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:16709 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:55,676 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:17132 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:55,776 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:17592 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:55,812 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:17719 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:55,948 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:18205 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:55,988 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:18377 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:56,150 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:18991 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:56,177 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:19259 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:56,293 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:19538 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:56,320 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:19647 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:56,412 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:20888 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:56,487 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:21167 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:56,559 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:21430 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:56,687 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:22250 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:56,729 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:22466 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:56,783 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:22681 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:56,811 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:22788 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,056 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:23718 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,105 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:23893 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,160 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:24104 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:57,184 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:24200 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,223 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:24796 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,282 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:25024 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,298 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:25073 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,419 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:25624 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,636 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:26360 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:57,654 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:26423 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,837 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:27314 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:57,838 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:27315 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,859 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:27409 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:57,981 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:27853 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:57,988 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:27877 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:58,031 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:28012 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:58,063 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:28131 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:58,130 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:28380 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:58,173 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:28472 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:58,178 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:28484 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:58,253 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:28727 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:58,303 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:28837 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:58,541 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:30132 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:58,549 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:30166 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:58,597 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:30731 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:58,680 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:31237 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:58,690 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:31286 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:58,694 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:31300 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:58,792 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:31659 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,005 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:32456 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,095 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:32726 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,223 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:33173 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,253 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:33234 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,268 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:33274 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,299 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:33384 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,303 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:33396 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,420 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:33739 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,435 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:33803 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,446 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:33840 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,508 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:34075 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,525 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:34132 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,533 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:34172 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,576 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:34324 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,579 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:34330 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,590 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:34374 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,647 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:35004 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,648 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:35008 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,659 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:35048 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,708 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:35720 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,778 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:35935 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,815 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:36097 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,830 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:36146 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,889 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:36340 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:16:59,937 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:36522 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:16:59,957 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:36607 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,064 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:36977 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,148 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:37275 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:00,158 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:37307 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,187 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:37409 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,259 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:37540 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,483 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:39211 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,538 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:39874 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:00,543 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:39891 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,608 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:40122 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,784 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:40646 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,842 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:40864 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:00,864 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:40957 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:00,895 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:41074 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,151 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:45868 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,198 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:45944 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,270 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46197 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,281 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46232 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:01,305 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46326 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,359 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46446 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,390 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46552 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:01,392 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46557 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:01,396 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46570 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,505 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46748 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:01,519 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46801 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,617 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:46997 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,759 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:48114 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:01,788 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:48232 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:01,934 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:49113 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:01,940 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:49122 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,055 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:49593 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,085 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:49727 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,144 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:49963 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,211 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:50202 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,346 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:52998 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:02,366 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:53081 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,415 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:53269 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:02,417 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:53270 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,504 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:53501 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,702 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:54294 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:02,718 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:54348 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,737 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:54418 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:02,821 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:55567 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:03,198 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:58068 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:03,234 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:58779 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:03,283 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:58883 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:03,369 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:59663 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:03,440 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:59821 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:03,478 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:60011 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:03,616 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:60611 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:03,728 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:61055 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:03,954 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:63106 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:03,958 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:63117 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,029 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:63380 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:04,040 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:63424 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,103 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:63563 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:04,126 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:64271 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,152 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:64388 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:04,162 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:64406 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,235 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:64631 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:04,244 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:64656 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,312 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:64876 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,510 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:66484 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:04,551 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:66571 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:04,555 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:66587 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,610 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:67214 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,694 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:67548 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:04,699 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:67571 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,780 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:67870 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,835 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:68101 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:04,856 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:68192 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,898 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:68344 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:04,899 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:68345 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:04,942 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:68520 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,066 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:73971 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:05,073 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:73992 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,159 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:74488 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,211 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:74917 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,264 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:75055 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,376 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:75863 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,468 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:76233 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:05,473 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:76246 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,528 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:76451 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,570 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:76602 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,628 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:76810 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,680 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:76998 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,745 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:77247 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,776 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:77343 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,882 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:78017 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:05,992 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:78426 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,052 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:78631 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,095 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:78763 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,124 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:78854 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,222 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:79191 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,259 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:79342 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,310 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:79532 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,391 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:80040 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,426 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:80107 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,440 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:80154 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,459 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:81046 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,546 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:81397 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,600 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:81606 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,619 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:81635 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,660 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:81749 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,679 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:81811 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,693 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:81846 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,698 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:81850 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,717 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:81882 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,784 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:82003 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,787 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:82009 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,832 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:82122 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,930 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:82477 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,948 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:82547 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:06,980 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:82663 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:06,987 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:82697 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:07,131 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:83601 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:07,213 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:84112 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:07,273 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:84618 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:07,309 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:84763 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:07,315 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:84779 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:07,352 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:84927 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:07,669 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:88194 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:07,766 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:88921 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:07,882 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:89247 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:07,921 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:89392 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:07,930 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:89434 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:07,951 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:89503 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:08,021 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:89701 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:08,027 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:89715 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:08,037 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:89759 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:08,092 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:90280 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:08,232 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:91390 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:08,428 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:92755 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:08,485 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:92940 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:08,645 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:93488 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:08,772 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:93825 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:08,896 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:94333 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:08,899 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:94341 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:08,931 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:94459 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:09,053 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:94915 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:09,123 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:97034 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:09,375 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:99838 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:09,445 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:100024 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:09,446 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:100025 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:09,483 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:100121 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:09,522 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:100211 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:09,793 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:102225 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:09,871 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:102436 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:09,911 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:102581 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,034 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:104813 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,115 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:105567 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,202 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:105945 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,340 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:106434 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,404 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:106792 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:10,438 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:106919 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,491 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:107128 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,543 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:107320 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,583 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:107446 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,608 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:107538 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,672 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:107787 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,719 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:107964 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:10,724 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:107971 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:10,923 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:108746 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:11,053 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:109166 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:11,058 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:109179 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:11,062 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:109182 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:11,126 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:109657 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:11,188 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:109787 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:11,225 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:109891 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:11,400 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:110500 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:11,407 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:110517 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:11,409 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:110523 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO  04:17:11,438 TraversalEngine - mc1r_bac:110633        2.97e+05   60.2 s        3.4 m     85.9%        70.1 s     9.9 s "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:11,543 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:110962 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:11,784 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:112393 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:11,937 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:113535 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,005 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:115647 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,028 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:115658 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,042 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:115678 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,069 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:115777 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,111 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:115923 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,201 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:116287 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,202 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:116291 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,237 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:116418 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,240 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:116425 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,261 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:116503 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,312 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:116636 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,358 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:116793 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,393 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:116853 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,410 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:116914 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,461 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:117129 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,557 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:117472 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,578 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:117550 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,580 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:117554 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,662 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:117866 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,663 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:117873 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,669 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:117899 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,765 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:118233 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,779 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:118322 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,793 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:118377 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,840 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:118556 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,856 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:118621 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,879 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:118715 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:12,925 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:118874 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:12,931 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:118898 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,018 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:119254 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,052 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:119389 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:13,092 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:119408 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,115 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:119442 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:13,146 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:119554 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,244 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:119885 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,268 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:119945 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:13,276 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:119974 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:13,293 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:120054 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:13,302 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:120083 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,360 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:120292 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,411 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:120389 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,450 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:120505 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,502 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:120709 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,528 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:120800 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:13,538 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:120837 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,620 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:121120 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,655 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:121242 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,786 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:121647 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,800 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:121676 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:13,823 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:121769 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,864 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:121898 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:13,967 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:122266 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:13,982 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:122319 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,093 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:122742 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,099 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:122759 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,177 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:123047 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,216 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:123187 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,284 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:123415 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,317 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:123535 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:14,329 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:123589 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,404 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:123844 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,460 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:124229 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,581 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:124771 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:14,613 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:124884 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,657 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:125066 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,700 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:125247 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:14,703 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:125263 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,727 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:125345 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,824 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:126096 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:14,892 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:126396 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,083 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:127371 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:15,095 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:127419 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,156 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:127921 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,204 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:128092 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:15,219 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:128158 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,518 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:129628 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:15,526 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:129654 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,581 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:129856 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:15,586 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:129875 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,599 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:129920 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,679 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:130220 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:15,690 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:130267 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,700 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:130301 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:15,719 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:130374 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:15,729 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:130406 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,770 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:130543 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,827 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:130784 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:15,830 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:130797 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:15,872 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:130958 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,026 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:131586 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:16,044 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:131663 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,065 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:131735 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:16,088 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:131820 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,140 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:132033 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:16,143 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:132046 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,250 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:132462 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:16,254 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:132477 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,316 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:132729 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,403 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:133040 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,508 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:133419 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,586 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:133681 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:16,603 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:133728 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:16,614 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:133762 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,660 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:133935 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,706 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:134084 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,852 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:134524 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,932 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:134843 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:16,997 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:135090 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:17,006 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:135111 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:17,072 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:135309 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:17,239 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:136297 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:17,240 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:136303 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:17,310 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:136574 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:17,331 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:136655 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:17,421 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:137029 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:17,451 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:137422 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:17,455 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:137441 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:17,588 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:137996 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:17,621 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:138121 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:17,631 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:138158 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:17,656 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:138281 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:17,828 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:139162 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:17,875 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:139842 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:17,898 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:140114 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:18,035 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:140661 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:18,069 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:140802 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:18,078 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:140836 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:18,113 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:140971 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:18,120 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:140999 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:18,204 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:141312 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:18,214 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:141354 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:18,292 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:141750 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:18,454 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:142782 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:18,467 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:142834 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:18,483 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:142892 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:18,971 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:147782 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:18,988 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:147897 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,151 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:149032 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,197 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:149206 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,228 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:149319 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,281 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:149445 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,320 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:149587 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:19,338 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:149644 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,436 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:150147 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,480 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:150294 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:19,494 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:150342 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,524 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:150437 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,697 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:151009 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:19,721 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:151255 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,746 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:151347 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:19,766 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:152632 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,812 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:154064 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:19,895 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:155177 has 3 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:20,063 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:155618 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument ", 
        "WARN  04:17:20,065 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:155621 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:20,148 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:155970 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:20,215 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:156243 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:20,307 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:157267 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:20,539 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:158240 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "WARN  04:17:20,618 ExactAFCalculationModel - this tool is currently set to genotype at most 1 alternate alleles in a given context, but the context at mc1r_bac:158572 has 2 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO  04:17:20,792 TraversalEngine - Total runtime 69.57 secs, 1.16 min, 0.02 hours ", 
        "INFO  04:17:20,792 TraversalEngine - 8412 reads were filtered out during traversal out of 158595 total (5.30%) ", 
        "INFO  04:17:20,792 TraversalEngine -   -> 8412 reads (5.30% of total) failing UnmappedReadFilter "
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "INFO  04:17:23,136 GATKRunReport - Uploaded run statistics report to AWS S3 "
       ]
      }
     ], 
     "prompt_number": 251
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "afli = map(float,re.findall('AF=([\\d\\.]+)',open(vcf).read()))", 
      "len(afli)"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "pyout", 
       "prompt_number": 252, 
       "text": [
        "198424"
       ]
      }
     ], 
     "prompt_number": 252
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "hist(array(afli)*num_haplo,bins=range(1,num_haplo+1))", 
      "figure(2)", 
      "h,b = histogram(array(afli)*num_haplo,bins=range(0,num_haplo+1))", 
      "plot(b[1:-1],(array(h,dtype=float)/sum(h))[1:],'--r')", 
      "plot(b[1:-1],sfs[1:],'g')"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "pyout", 
       "prompt_number": 253, 
       "text": [
        "[<matplotlib.lines.Line2D at 0x10ff7e4d0>]"
       ]
      }, 
      {
       "output_type": "display_data", 
       "png": "iVBORw0KGgoAAAANSUhEUgAAAX8AAAD9CAYAAABUS3cAAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAG0FJREFUeJzt3V9MW+f9x/G3+aUQdXGqlggSibgZLcI4TYJpbCNVARdV\nG2NqYfQCqpFGhUktycaaf8oqoo7cNMrWjgBaCBflZh3johJSMy0hI5tpyITtrARV4CBYEpFNaSj9\npYtR0ym053fBen5NQ/n/x8n5vCRL5vF5nvN9HOuTcx4f2zbDMAxERMRS4pa7ABERWXoKfxERC1L4\ni4hYkMJfRMSCFP4iIhak8BcRsaAZhf8XX3yB2+3m2WefBSAajVJYWIjD4aCoqIixsTFz2/r6etLS\n0nC5XHR1dZntkUiErKwsUlNTqa6uXuBpiIjIbMwo/Ovq6nC5XNhsNgAaGxtxOBwMDg6SkpLC8ePH\nARgZGeHYsWOcOXOGxsZGqqqqzDH27t3LgQMHCIfDdHZ2cv78+UWYjoiIzMS04f/Pf/6TP/3pT/zk\nJz/hq8+DhUIhKioqSEhIoLy8nGAwCEAwGCQ/Px+Hw0Fubi6GYZhnBQMDA5SUlJCYmEhxcbHZR0RE\nlt604b97925+/etfExf3/5uGw2GcTicATqeTUCgETIR/RkaGuV16ejrBYJChoSGSkpLMdpfLRXd3\n94JNQkREZmfFVA/+8Y9/JCkpCbfbTSAQMNtn840QXy0Vfd1U/SfbXkREpjbbb+qZ8sj/b3/7G++9\n9x7f/e53eeGFF/jLX/7C9u3b8Xg8RCIRYOKNXI/HA4DP56O/v9/sf/HiRTweD48//jjXr1832/v7\n+8nOzp5yErrN//bLX/5y2Wu4n256PvV8xuptLqYM/zfeeIOrV69y+fJlWltbycvL43e/+x0+n4/m\n5mZu3bpFc3OzGeRer5f29naGh4cJBALExcVht9uBieWh1tZWRkdHaWtrw+fzzang6axe/Qg2m21e\nt9WrH1mU2kREYsWsrvP/akmmsrKS4eFh0tPT+de//sUrr7wCQHJyMpWVleTl5bFz507q6urMvm++\n+Sa/+tWv8Hg8bNu2ja1bty7gNP5fNHoDMOZ1mxhDROT+ZTPmes6wSGw225xPY77qPxHi86piXjXE\nikAggN/vX+4y7ht6PheWns+FM5fcVPhPPsp9Ef4iYg1zyU19vYOIiAUp/EVELEjhLyJiQQp/EREL\nUviLiFiQwl9ExIIU/iIiFqTwFxGxIIW/iIgFKfxFRCxI4S8iYkEKfxERC1L4i4hYkMJfRMSCFP4i\nIhak8BcRsSCFv4iIBSn8RUQsaMrw//zzz/H5fGRmZpKdnU1tbS0ANTU1pKSk4Ha7cbvdnDx50uxT\nX19PWloaLpeLrq4usz0SiZCVlUVqairV1dWLNB0REZmJaX/D97PPPuPBBx/kP//5D08++SRtbW20\ntLRgt9vZs2fPHduOjIyQk5PD6dOnuXz5Mrt37+aDDz4AoKCggB07dvDMM89QWFjI0aNH2bp1690F\n6Td8RURmZVF+w/fBBx8EYGxsjPHxcRISEgAm3VEwGCQ/Px+Hw0Fubi6GYTA2NgbAwMAAJSUlJCYm\nUlxcTDAYnFWhIiKycKYN/y+//JItW7aQnJzMT3/6UxwOBwANDQ1kZ2dz5MgRotEoAKFQiIyMDLNv\neno6wWCQoaEhkpKSzHaXy0V3d/dCz0VERGZoxXQbxMXF0dvby5UrVygoKOCpp56isrKS119/nZs3\nb7J//36amprYt2/fpGcDE8swd5ru9KSmpsa87/f78fv9089ERMQiAoEAgUBgXmNMu+b/dfv27ePx\nxx/nlVdeMdt6e3vZuXMn586d48SJE3R0dFBXVwdAZmYmZ8+exW63k5qayqVLlwB46623WLlyJbt2\n7bq7IK35i4jMyoKv+Y+OjvLpp58C8Mknn3D69GkKCwu5du0aAOPj47S0tFBQUACA1+ulvb2d4eFh\nAoEAcXFx2O12AJxOJ62trYyOjtLW1obP55v1BEVEZGFMuexz7do1duzYwRdffMHatWvZt28f69at\n48UXX+TChQvEx8eTk5NDZWUlAMnJyVRWVpKXl0d8fDxNTU3mWG+++SZlZWW89tprlJaWTnqlj4iI\nLI1ZLfssBS37iIjMzqJc6ikiIvcfhb+IiAUp/EVELEjhLyJiQQp/ERELUviLiFiQwl9ExIIU/iIi\nFqTwFxGxIIW/iIgFKfxFRCxI4S8iYkEKfxERC1L4i4hYkMJfRMSCFP4iIhak8BcRsSCFv4iIBSn8\nRUQsaMrw//zzz/H5fGRmZpKdnU1tbS0A0WiUwsJCHA4HRUVFjI2NmX3q6+tJS0vD5XLR1dVltkci\nEbKyskhNTaW6unqRpiMiIjMxZfivXLmSv/71r1y4cIHOzk7efvttBgcHaWxsxOFwMDg4SEpKCseP\nHwdgZGSEY8eOcebMGRobG6mqqjLH2rt3LwcOHCAcDtPZ2cn58+cXd2YiIvKtpl32efDBBwEYGxtj\nfHychIQEQqEQFRUVJCQkUF5eTjAYBCAYDJKfn4/D4SA3NxfDMMyzgoGBAUpKSkhMTKS4uNjsIyIi\nS2/FdBt8+eWXuN1u+vr6OHr0KA6Hg3A4jNPpBMDpdBIKhYCJ8M/IyDD7pqenEwwGefTRR0lKSjLb\nXS4Xv//979m1a9ek+6ypqTHv+/1+/H7/XOYmInJfCgQCBAKBeY0xbfjHxcXR29vLlStXKCgo4Kmn\nnsIwjBnvwGaz3dU2Xf+vh7+IiNzpmwfFhw4dmvUYM77aZ8OGDRQUFBAMBvF4PEQiEWDijVyPxwOA\nz+ejv7/f7HPx4kU8Hg+PP/44169fN9v7+/vJzs6edbEiIrIwpgz/0dFRPv30UwA++eQTTp8+TWFh\nIT6fj+bmZm7dukVzc7MZ5F6vl/b2doaHhwkEAsTFxWG324GJ5aHW1lZGR0dpa2vD5/Mt8tREROTb\n2Iwp1mA+/PBDduzYwRdffMHatWv58Y9/zIsvvkg0GqWsrIyenh6ysrJ45513WLVqFQB1dXU0NDQQ\nHx9PU1MT27ZtAyaO9svKyrhx4walpaUcPnx48oJstlktK03WH+be/7+jzKsGEZGlNJfcnDL8l4PC\nX0RkduaSm/qEr4iIBSn8RUQsSOEvImJBCn8REQtS+IuIWJDCX0TEghT+IiIWpPAXEbEghb+IiAUp\n/EVELEjhLyJiQQp/ERELUviLiFiQwl9ExIIU/iIiFqTwFxGxIIW/iIgFKfxFRCxoyvC/evUqTz/9\nNBs3bsTv99PS0gJATU0NKSkpuN1u3G43J0+eNPvU19eTlpaGy+Wiq6vLbI9EImRlZZGamkp1dfUi\nTUdERGZiyt/w/eijj/joo4/IzMxkdHQUr9dLb28vv/nNb7Db7ezZs+eO7UdGRsjJyeH06dNcvnyZ\n3bt388EHHwBQUFDAjh07eOaZZygsLOTo0aNs3br17oL0G74iIrMyl9xcMdWDa9euZe3atQCsWbOG\njRs3Eg6HASbdUTAYJD8/H4fDgcPhwDAMxsbGWLVqFQMDA5SUlABQXFxMMBicNPxFRGTxzXjNf2ho\niL6+Pnw+HwANDQ1kZ2dz5MgRotEoAKFQiIyMDLNPeno6wWCQoaEhkpKSzHaXy0V3d/dCzUFERGZp\nyiP/r0SjUUpKSqitreU73/kOlZWVvP7669y8eZP9+/fT1NTEvn37Jj0bmFiGudN0pyc1NTXmfb/f\nj9/vn0mZIiKWEAgECAQC8xpjyjV/gNu3b/PDH/6QgoICXn311bse7+3tZefOnZw7d44TJ07Q0dFB\nXV0dAJmZmZw9exa73U5qaiqXLl0C4K233mLlypXs2rXr7oK05i8iMitzyc0pl30Mw6CiooInnnji\njuC/du0aAOPj47S0tFBQUACA1+ulvb2d4eFhAoEAcXFx2O12AJxOJ62trYyOjtLW1mYuH4mIyNKb\n8si/q6uLnJwcNm/ebC7fvPHGG/zhD3/gwoULxMfHk5OTw8GDB3nkkUcAqKuro6Ghgfj4eJqamti2\nbRsA/f39lJWVcePGDUpLSzl8+PDkBenIX0RkVuaSm9Mu+yw1hb+IyOws+LKPiIjcnxT+IiIWpPAX\nEbEghb+IiAUp/EVELEjhLyJiQQp/ERELUviLiFiQwl9ExIIU/iIiFqTwFxGxIIW/iIgFKfxFRCxI\n4S8iYkEKfxERC1L4i4hYkMJfRMSCFP4iIhak8BcRsaApw//q1as8/fTTbNy4Eb/fT0tLCwDRaJTC\nwkIcDgdFRUWMjY2Zferr60lLS8PlctHV1WW2RyIRsrKySE1Npbq6epGmIyIiMzFl+D/wwAPU1tbS\n19fHu+++y8GDB4lGozQ2NuJwOBgcHCQlJYXjx48DMDIywrFjxzhz5gyNjY1UVVWZY+3du5cDBw4Q\nDofp7Ozk/PnzizszERH5VlOG/9q1a8nMzARgzZo1bNy4kXA4TCgUoqKigoSEBMrLywkGgwAEg0Hy\n8/NxOBzk5uZiGIZ5VjAwMEBJSQmJiYkUFxebfWLTCmw227xuq1c/styTEBH5VjNe8x8aGqKvrw+v\n10s4HMbpdALgdDoJhULARPhnZGSYfdLT0wkGgwwNDZGUlGS2u1wuuru7F2oOi2AcMOZ1i0ZvLH3Z\nIiIztGImG0WjUUpKSqitrWXVqlUYhjHjHdhstrvaputfU1Nj3vf7/fj9/hnvT0TkfhcIBAgEAvMa\nY9rwv337Ns8//zzbt2+nsLAQAI/HQyQSwe12E4lE8Hg8APh8Pjo6Osy+Fy9exOPxYLfbuX79utne\n399Pdnb2t+7z6+EvIiJ3+uZB8aFDh2Y9xpTLPoZhUFFRwRNPPMGrr75qtvt8Ppqbm7l16xbNzc1m\nkHu9Xtrb2xkeHiYQCBAXF4fdbgcmlodaW1sZHR2lra0Nn88362JFRGRh2Iwp1mC6urrIyclh8+bN\n5vLN4cOHeeqppygrK6Onp4esrCzeeecdVq1aBUBdXR0NDQ3Ex8fT1NTEtm3bgImj/bKyMm7cuEFp\naSmHDx+evCCbbVbLSpP1n1h3n4+FGWM+8xARmam55OaU4b8cFP4iIrMzl9zUJ3xFRCxI4S8iYkEK\nfxERC1L4i4hYkMJfRMSCFP4iIhak8BcRsSCFv4iIBSn8RUQsSOEvImJBCn8REQtS+IuIWJDCX0TE\nghT+IiIWpPAXEbEghb+IiAUp/EVELEjhLyJiQQp/ERELmjL8y8vLSU5OZtOmTWZbTU0NKSkpuN1u\n3G43J0+eNB+rr68nLS0Nl8tFV1eX2R6JRMjKyiI1NZXq6upFmIaIiMzGlOH/0ksvcerUqTvabDYb\ne/bsoaenh56eHn7wgx8AMDIywrFjxzhz5gyNjY1UVVWZffbu3cuBAwcIh8N0dnZy/vz5RZiKiIjM\n1JThv23bNh5++OG72if7lfhgMEh+fj4Oh4Pc3FwMw2BsbAyAgYEBSkpKSExMpLi4mGAwuEDli4jI\nXMxpzb+hoYHs7GyOHDlCNBoFIBQKkZGRYW6Tnp5OMBhkaGiIpKQks93lctHd3T3PskVEZD5WzLZD\nZWUlr7/+Ojdv3mT//v00NTWxb9++Sc8GbDbbXW2TbfdNNTU15n2/34/f759tmSIi961AIEAgEJjX\nGDZjmjS+cuUKzz77LB9++OFdj/X29rJz507OnTvHiRMn6OjooK6uDoDMzEzOnj2L3W4nNTWVS5cu\nAfDWW2+xcuVKdu3aNXlBNtuM/oP41gnZbMDc+/93lAUZYz7zEBGZqbnk5qyXfa5duwbA+Pg4LS0t\nFBQUAOD1emlvb2d4eJhAIEBcXBx2ux0Ap9NJa2sro6OjtLW14fP5ZrtbERFZQFMu+7zwwgt0dnYy\nOjrK+vXrOXToEIFAgAsXLhAfH09OTg6VlZUAJCcnU1lZSV5eHvHx8TQ1NZnjvPnmm5SVlfHaa69R\nWlrK1q1bF3dWIiIypWmXfZaaln1ERGZnSZZ9RETk3qfwFxGxIIW/iIgFKfxFRCxI4S8iYkEKfxER\nC1L4i4hYkMJfRMSCFP4iIhak8BcRsSCFv4iIBSn8RUQsSOEvImJBCn8REQtS+IuIWJDCX0TEghT+\nIiIWpPAXEbEghb+IiAVNGf7l5eUkJyezadMmsy0ajVJYWIjD4aCoqIixsTHzsfr6etLS0nC5XHR1\ndZntkUiErKwsUlNTqa6uXoRpiIjIbEwZ/i+99BKnTp26o62xsRGHw8Hg4CApKSkcP34cgJGREY4d\nO8aZM2dobGykqqrK7LN3714OHDhAOByms7OT8+fPL8JURERkpqYM/23btvHwww/f0RYKhaioqCAh\nIYHy8nKCwSAAwWCQ/Px8HA4Hubm5GIZhnhUMDAxQUlJCYmIixcXFZh8REVkes17zD4fDOJ1OAJxO\nJ6FQCJgI/4yMDHO79PR0gsEgQ0NDJCUlme0ul4vu7u751i0iIvOwYrYdDMOY8bY2m21O/Wtqasz7\nfr8fv98/432KiNzvAoEAgUBgXmPMOvw9Hg+RSAS3200kEsHj8QDg8/no6Ogwt7t48SIejwe73c71\n69fN9v7+frKzs6fcx9fDX0RE7vTNg+JDhw7NeoxZL/v4fD6am5u5desWzc3NZpB7vV7a29sZHh4m\nEAgQFxeH3W4HJpaHWltbGR0dpa2tDZ/PN+tCRURkARlTKC0tNdatW2fEx8cbKSkpRnNzs3Hz5k3j\nueeeM9avX28UFhYa0WjU3P7o0aPGY489ZmRkZBjvv/++2d7X12e43W5jw4YNxi9+8YupdmlMU9K0\nAAOMed4WZgwRkaUwl7yx/bdjzLDZbLN6X2Gy/jDfKS3MGDH21IrIfWouualP+IqIWJDCX0TEghT+\nIiIWpPAXEbEghb+IiAUp/EVELEjhLyJiQQp/ERELUvgvmhXYbLY531avfmS5JyAi9zF9wnfyUWJg\nDH1CWERmRp/wFRGRGVH4i4hYkMJfRMSCFP4iIhak8BcRsSCFv4iIBSn8RUQsSOEvImJBCn8REQua\nc/hv2LCBzZs343a78Xq9AESjUQoLC3E4HBQVFTE2NmZuX19fT1paGi6Xi66urvlXLiIiczbn8LfZ\nbAQCAXp6egiFQgA0NjbicDgYHBwkJSWF48ePAzAyMsKxY8c4c+YMjY2NVFVVLUz1IiIyJ/Na9vnm\nd0mEQiEqKipISEigvLycYDAIQDAYJD8/H4fDQW5uLoZhEI1G57NrERGZh3kd+efl5VFUVMR7770H\nQDgcxul0AuB0Os0zgmAwSEZGhtk3PT3dfExERJbeirl2PHfuHOvWrSMSifDss8/i9Xpn9a1yE9++\nObmamhrzvt/vx+/3z7VMEZH7TiAQIBAIzGuMBflK5z179pCRkcGpU6c4ePAgbrebv//97xw+fJh3\n332XEydO0NHRQV1dHQCZmZmcPXsWu91+d0H6Smezv77SWURmYsm+0vmzzz4z1+w//vhj2tvbyc/P\nx+fz0dzczK1bt2hubiY7OxsAr9dLe3s7w8PDBAIB4uLiJg1+ERFZGnNa9rl+/To/+tGPAEhMTGTv\n3r2sX7+eyspKysrKSE9PJysriyNHjgCQnJxMZWUleXl5xMfH09TUtHAzEBGRWdMveU0+SgyMoWUf\nEZkZ/ZKXiIjMiMJfRMSCFP4iIhY05+v8ZbGtmPKzEDNhtz/MzZv/u0D1iMj9RG/4Tj5KDIyxMDXE\n2D+viCwCveErIiIzovAXEbEghb+IiAUp/EVELEjhLyJiQQp/EREL0nX+9zV9VkBEJqfr/CcfJQbG\niIUaJsaIsZeIiHyDrvMXEZEZUfiLiFiQwl9ExIL0hq9MQ28ai9yPdOQv0xhn4k3jud+i0RtLX/Yi\nWL36EWw227xuq1c/cs/XIPcHXe0z+SgxMEYs1LBQYzzAxH8i8+l/e14VLMTZx0K9tpb/9T3ffw+d\nzS201asfmfdB0mxfV0sa/u+//z4vv/wy4+PjVFVV8bOf/ezughT+MVRDrIwRG4E3IRbqWO5/D1iY\nefwP8MU8a5jfQUHsjAFL/ZvfSxr+breburo6Hn30Ub7//e/T1dXFmjVr7ixI4R9DNcTKGLFQQ6yM\nEQs1xMoYsVBDrIwRw9f5//vf/wYgJyeHRx99lO9973sEg8Gl2r2IiHzNkoV/OBzG6XSaf7tcLrq7\nu5dq9yIi8jUxeannfC8tnDiFmncVMTBGLNQQK2PEQg2xMkYs1BArY8RCDbE0xswtWfh7PB72799v\n/t3X10d+fv5d28XYxUciIvelJVv2eeihh4CJK36uXLnCn//8Z3w+31LtXkREvmZJl32OHj3Kyy+/\nzO3bt6mqqrrrSh8REVkaS/oJ39zcXCKRCENDQ1RVVd3x2Pvvv09GRgZpaWk0NDQsZVn3nQ0bNrB5\n82bcbjder3e5y7nnlJeXk5yczKZNm8y2aDRKYWEhDoeDoqIixsbGlrHCe8tkz2dNTQ0pKSm43W7c\nbjenTp1axgrvHVevXuXpp59m48aN+P1+WlpagLm9PmPm6x1+/vOf09TUREdHB7/97W8ZHR1d7pLu\nWTabjUAgQE9PD6FQaLnLuee89NJLd4VRY2MjDoeDwcFBUlJSOH78+DJVd++Z7Pm02Wzs2bOHnp4e\nenp6Jn3/T+72wAMPUFtbS19fH++++y4HDx4kGo3O6fUZE+GvzwAsPL1xPnfbtm3j4YcfvqMtFApR\nUVFBQkIC5eXlen3OwmTPJ+g1Ohdr164lMzMTgDVr1rBx40bC4fCcXp8xEf76DMDCstls5OXlUVRU\nxHvvvbfc5dwXvv4adTqdOqNaAA0NDWRnZ3PkyBGi0ehyl3PPGRoaoq+vD6/XO6fXZ0yEvyysc+fO\n0dvby+HDh9mzZw8fffTRcpd0z9NR6sKqrKzk8uXLtLe3849//IOmpqblLumeEo1GKSkpoba2llWr\nVs3p9RkT4e/xeLh48aL5d19fH9nZ2ctY0b1t3bp1AGRkZPDcc89x4sSJZa7o3ufxeIhEIgBEIhE8\nHs8yV3RvS0pKwmaz8dBDD7Fr1y7a2tqWu6R7xu3bt3n++efZvn07hYWFwNxenzER/voMwML57LPP\nzFPojz/+mPb2dr2ZtgB8Ph/Nzc3cunWL5uZmHZzM07Vr1wAYHx+npaWFgoKCZa7o3mAYBhUVFTzx\nxBO8+uqrZvucXp9GjAgEAobT6TQee+wxo66ubrnLuWddunTJ2LJli7FlyxYjLy/PePvtt5e7pHtO\naWmpsW7dOiM+Pt5ISUkxmpubjZs3bxrPPfecsX79eqOwsNCIRqPLXeY946vn84EHHjBSUlKMt99+\n29i+fbuxadMm48knnzR2795tfPLJJ8td5j3h7Nmzhs1mM7Zs2WJkZmYamZmZxsmTJ+f0+oy5H3MR\nEZHFFxPLPiIisrQU/iIiFqTwFxGxIIW/iIgFKfxFRCxI4S8iYkH/B3z2cqdlhTeIAAAAAElFTkSu\nQmCC\n"
      }, 
      {
       "output_type": "display_data", 
       "png": "iVBORw0KGgoAAAANSUhEUgAAAYIAAAD9CAYAAACx+XApAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAIABJREFUeJzt3X1cE3e+L/DPhCcfUCSCjwgKsoaoFeQhVClNbYuubkVa\ndl229uxVzx6Krdq6uL2e01vpnrPbdbsWK8dq9h7p3r2Ybu/W2lrvVkW3iNI1QUXXIrSCUm1Lu4CA\nQUETMuePaDQSwlNwAvN5v155JZP5zeQ7Mc6HmfnNjCCKoggiIpIthdQFEBGRtBgEREQyxyAgIpI5\nBgERkcwxCIiIZI5BQEQkc10GQXFxMaKiohAZGYm8vDynbTZs2IDw8HDExsaisrISANDW1gaNRoPo\n6GgkJiYiNzfX3t5kMiE1NRWhoaFYsmQJWlpa3LQ4RETUU10Gwdq1a6HT6XDo0CFs27YN9fX1DuON\nRiOOHj2KEydOIDs7G9nZ2QCAIUOG4JNPPsHp06dx5MgR7Ny5E1VVVQCA7du3IzQ0FOfPn0dISAh2\n7NjRD4tGRETd4TIImpubAQDJyckICwtDSkoKDAaDQxuDwYD09HQolUpkZGSgoqLCPm7YsGEAgJaW\nFlgsFvj5+QGwhcfKlSvh5+eHFStWdJgnERHdPy6DoLS0FCqVyj6sVqtx/PhxhzZGoxFqtdo+HBwc\njOrqagBAe3s7Zs2ahbFjx+L555/HpEmTOsxXpVLBaDS6Z2mIiKjHvPs6A1EUce9VKgRBAAB4eXnh\nzJkzqKmpwcKFCzF37lzExMR0aN+Z2/MhIqLu6+mVg1xuEcTHx9sP/gJAeXk5EhMTHdpoNBqcO3fO\nPlxXV4fw8HCHNpMnT8bChQvtf/nHx8fbdyFVVFQgPj6+0xpuBw0ffXts3LhR8hoG04PfJ79PT330\nhssgCAgIAGDrOVRTU4PCwkJoNBqHNhqNBrt370ZDQwP0ej2ioqIAAPX19WhqagIANDQ04ODBg1i8\neLF9mvz8fLS2tiI/P79DuBAR0f3T5a6hLVu2IDMzE2azGWvWrEFQUBB0Oh0AIDMzEwkJCUhKSkJc\nXByUSiUKCgoAALW1tfjpT3+K9vZ2jBs3DtnZ2Rg/fjwAICsrC8uWLcO0adMwe/ZsbNq0qR8XkYiI\nXBHE3m5L3AeCIPR6U4ccFRUVQavVSl3GoMHv0734fbpPb9abDAIiokGkN+tNXmKCiEjmGARERDLH\nICAikjkGARGRzDEIiIhkjkFARCRzDAIiIpljEBARyRyDgIhI5hgEREQyxyAgIpI5BgERkcwxCIiI\nZI5BQEQkcwwCIiKZG7RB8NXVr/DG396QugwiIo83aIPAKlqx+W+bpS6DiMjjDdogmDhiIhquN+C6\n+brUpRARebRBGwReCi9MHjUZFxovSF0KEZFHG7RBAABTlVNRfaVa6jKIiDzaoA6CCGUEqhsZBERE\nrgzuIAiMQNWVKqnLICLyaIM6CKYqp3KLgIioC4M6CCICI3iMgIioC4M6CCaPmozLVy/D3G6WuhQi\nIo81qIPAz9sP4/3H41LzJalLISLyWIM6CAD2HCIi6kqXQVBcXIyoqChERkYiLy/PaZsNGzYgPDwc\nsbGxqKysBABcvnwZjzzyCKZPnw6tVgu9Xm9vn5OTg5CQEMTExCAmJgb79+930+J0xJ5DRESueXfV\nYO3atdDpdAgLC8P8+fORkZGBoKAg+3ij0YijR4/ixIkTOHDgALKzs7Fv3z74+PggNzcX0dHRqK+v\nR0JCAhYvXgx/f38IgoB169Zh3bp1/bpwAHsOERF1xeUWQXNzMwAgOTkZYWFhSElJgcFgcGhjMBiQ\nnp4OpVKJjIwMVFRUAADGjRuH6OhoAEBQUBCmT5+O0tJS+3SiKLp1QTrDnkNERK65DILS0lKoVCr7\nsFqtxvHjxx3aGI1GqNVq+3BwcDCqqx1XvFVVVSgvL0dCQoL9vby8PCQmJmLTpk0wmUx9WghXIpTc\nNURE5EqXu4a6Iopih7/uBUGwvzaZTFi6dClyc3MxfPhwAEBWVhZeeeUVXL16FevXr4dOp0N2drbT\n+efk5Nhfa7VaaLXaHtUXERiBC40XIIqiQ11ERINBUVERioqK+jQPQXSxj6a5uRlarRZlZWUAgNWr\nV2PBggVYtGiRvU1eXh4sFgtefPFFAEBERIR9i8BsNmPRokVYuHAhXnjhBaefcebMGaxatQolJSUd\nixMEt+xCGvu7sSjLLMOEERP6PC8iIk/Wm/Wmy11DAQEBAGw9h2pqalBYWAiNRuPQRqPRYPfu3Who\naIBer0dUVBQA25bCypUrMWPGjA4hUFtbCwCwWCzQ6/VYuHBhj4ruKfYcIiLqXJe7hrZs2YLMzEyY\nzWasWbMGQUFB0Ol0AIDMzEwkJCQgKSkJcXFxUCqVKCgoAACUlJSgoKAADzzwAGJiYgAAr732GhYs\nWICXXnoJp0+fhq+vL5KTk5GVldWPi3jnctTJYcn9+jlERAORy11DUnPXrqFXi16F2WrGf8z7DzdU\nRUTkudy+a2iwYM8hIqLOySIIeFIZEVHnZBEEPKmMiKhzsgiCoGFBsFgtuNJ6RepSiIg8jiyCQBAE\n3sieiKgTsggCgJejJiLqjHyCgCeVERE5JZsgYM8hIiLnZBME7DlEROScfIKAJ5URETklmyAIGRmC\nxrZGXDdfl7oUIiKPIpsgUAgKTB41GRcaL0hdChGRR5FNEADsOURE5IysgoAnlRERdSSrIIgI5Ell\nRET3klcQsOcQEVEHsgoCnlRGRNSRrIJg8qjJ+OrqVzC3m6UuhYjIY8gqCHy9fDHefzy+bP5S6lKI\niDyGrIIAYM8hIqJ7yS4IeDlqIiJH8gsCnlRGRORAdkHAnkNERI5kFwS8HDURkSPZBUF4YDguNF6A\nVbRKXQoRkUeQXRCM8BuBkX4jUWuqlboUIiKPILsgANhziIjobvIMAvYcIiKy6zIIiouLERUVhcjI\nSOTl5Tlts2HDBoSHhyM2NhaVlZUAgMuXL+ORRx7B9OnTodVqodfr7e1NJhNSU1MRGhqKJUuWoKWl\nxU2L0z3sOUREdEeXQbB27VrodDocOnQI27ZtQ319vcN4o9GIo0eP4sSJE8jOzkZ2djYAwMfHB7m5\nuSgvL8d7772Hl19+2b7C3759O0JDQ3H+/HmEhIRgx44d/bBonWPPISKiO1wGQXNzMwAgOTkZYWFh\nSElJgcFgcGhjMBiQnp4OpVKJjIwMVFRUAADGjRuH6OhoAEBQUBCmT5+O0tJSALbwWLlyJfz8/LBi\nxYoO8+xvvBw1EdEdLoOgtLQUKpXKPqxWq3H8+HGHNkajEWq12j4cHByM6mrHv7arqqpQXl6OhISE\nDvNVqVQwGo19W4oe4q4hIqI7vPs6A1EUIYqiw3uCINhfm0wmLF26FLm5uRg+fLh9mu7Kycmxv9Zq\ntdBqtd2bsKkJ2LgRePPNDqNGDx0Nq2jFldYrUA5VdrsWIiJPU1RUhKKior7NRHShqalJjI6Otg8/\n//zz4r59+xzabN26VXzjjTfsw+Hh4fbXN2/eFB9//HExNzfXYZonn3xSPHXqlCiKonjixAnxqaee\ncvr5XZTnWnu7KI4aJYrffut0dMyOGNHwlaH38yci8kC9WW+63DUUEBAAwNZzqKamBoWFhdBoNA5t\nNBoNdu/ejYaGBuj1ekRFRd0OGKxcuRIzZszACy+80GGa/Px8tLa2Ij8/H4mJiX1LM2cUCuDBB4GS\nEqejeTlqIiKbLnsNbdmyBZmZmXjsscewatUqBAUFQafTQafTAQASEhKQlJSEuLg4bN68Ga+//joA\noKSkBAUFBfjrX/+KmJgYxMTEYP/+/QCArKwsXLp0CdOmTcPXX3+NZ599tn+WLikJOHbM6SieVEZE\nZCPc2pTwSIIg9Oh4QgfFxcD69YCTXkn/deq/cOzSMfxhyR96P38iIg/Tm/Xm4D6zOD4e+Owz4Nq1\nDqPYc4iIyGZwB8HQocDhw4CPT4dRPKmMiMhmcAcBACQmAr6+Hd6eOHIiGtsace1mx60FIiI5GfxB\n0AmFoMCUUVNwofGC1KUQEUlKtkEAsOcQEREg9yDgcQIiIhkFgZPuVFOVU1HVyIvPEZG8ySMI9u4F\nnnmmw9vcIiAikksQTJ8OFBV12CrgMQIiIrkEQXg4YLEAly45vD151GR8dfUrmNvNEhVGRCQ9eQSB\nIDi97pCvly8mjJiAL5u/lKgwIiLpySMIAFsQOLkSKY8TEJHcyScI5s4FPv+8w9tTlVN520oikjX5\nBEFcHHDoUIe3IwJ5wJiI5E0+QSAItsc92HOIiOROPkHQCe4aIiK5k30QhAeG42LjRVhFq9SlEBFJ\nQvZB4O/rj5F+I1FrqpW6FCIiScgvCL74Aqh1XOlz9xARyZn8giAvD9i1y+EtHjAmIjmTXxA4OcOY\nXUiJSM7kFwRz59qC4K4L0HHXEBHJmfyCICQEGDHC4SxjXmaCiORMfkEA3NkquCVCGYGqK1UQndy8\nhohosJNnEPzoR0BgoH1w9NDRAIArrVekqoiISDLeUhcgicWLHQYFQbD3HBo9bLRERRERSUOeWwRO\n8DgBEckVg+AW9hwiIrnqMgiKi4sRFRWFyMhI5OXlOW2zYcMGhIeHIzY2FpWVlfb3V6xYgbFjx2Lm\nzJkO7XNychASEoKYmBjExMRg//79fVyMvuO5BEQkV10Gwdq1a6HT6XDo0CFs27YN9fX1DuONRiOO\nHj2KEydOIDs7G9nZ2fZxy5cvd7qSFwQB69atQ1lZGcrKyrBgwQI3LErf8OxiIpIrl0HQ3NwMAEhO\nTkZYWBhSUlJgMBgc2hgMBqSnp0OpVCIjIwMVFRX2cQ899BAC7+qdczeP6Kr5r/8KtLQA4K4hIpIv\nl0FQWloKlUplH1ar1Th+/LhDG6PRCLVabR8ODg5GdXXXf1nn5eUhMTERmzZtgslk6mnd7nHkCGA0\nAgAmjJiAprYmXLt5TZpaiIgk0ufuo6IodvjrXnByJ7C7ZWVl4ZVXXsHVq1exfv166HQ6h11Kd8vJ\nybG/1mq10Gq1fS35jtvXHZo3DwpBgSmjpuBC4wXMHDuz62mJiDxAUVERioqK+jYT0YWmpiYxOjra\nPvz888+L+/btc2izdetW8Y033rAPh4eHO4y/ePGiOGPGjE4/4/Tp0+KcOXOcjuuivL7bu1cUH3/c\nPviE/gnx/XPv9+9nEhH1o96sN13uGgoICABg6zlUU1ODwsJCaDQahzYajQa7d+9GQ0MD9Ho9oqKi\nugyf2lv3A7BYLNDr9Vi4cGHvUqyv5swBDAbAYgHAA8ZEJE9d7hrasmULMjMzYTabsWbNGgQFBUGn\n0wEAMjMzkZCQgKSkJMTFxUGpVKKgoMA+bUZGBo4cOYKGhgZMmjQJv/zlL7F8+XK89NJLOH36NHx9\nfZGcnIysrKz+W0JXRo8GJk4Ezp4FYmIQERiB8rpyaWohIpKIcGtTwiMJgtD/vYs+/RSYNg0YPRr7\nq/Zj8982o/CZwv79TCKiftKb9aY8rzV0tzlz7C95mQkikiNeYuIuYaPC8LXpa9xsvyl1KURE9w2D\n4C6+Xr6YOGIivmz6UupSiIjuGwbBPdhziIjkhkFwDx4nICK5YRAAQFUV8PDDAG5dc6iR1xwiIvlg\nEADA5MnAqVNAYyO3CIhIdhgEAODtDWg0wKef8hgBEckOg+C2uXOBY8cQERiBC40XYBWtUldERHRf\nMAhuu3Ul0uG+wzFqyCh8Y/pG6oqIiO4LBsFtiYnAuXOAxcLjBEQkKwyC20aMAGprAW9v3q2MiGSF\nQXA3X18AvJE9EckLg8AJ9hwiIjlhEDjBXUNEJCcMAiduHyz24Fs1EBG5DYPgXleuQPkPk+1l6xWJ\niyEi6n+8Mc293nsPQkkJps617R4aPWy01BUREfUrbhHc69aJZTxgTERywSC4l0oFNDUhwmcsTyoj\nIllgENxLoQDmzMHUf1h4OWoikgUGgTNJSYj4/B/cIiAiWWAQOPPYY4gYOZnHCIhIFgTRgzvLC4Ig\nWV9+q2iF/6/9Ube+DsN9h0tSAxFRT/Vmvcktgk4oBAWmBE7hVgERDXoMAhd4OWoikgMGgQtTlVO5\nRUBEgx6DwIWZY2bC+LVR6jKIiPpVl0FQXFyMqKgoREZGIi8vz2mbDRs2IDw8HLGxsaisrLS/v2LF\nCowdOxYzZ850aG8ymZCamorQ0FAsWbIELS0tfVyM/rHozHUcPL8fbZY2qUshIuo3XQbB2rVrodPp\ncOjQIWzbtg319fUO441GI44ePYoTJ04gOzsb2dnZ9nHLly/H/v37O8xz+/btCA0Nxfnz5xESEoId\nO3a4YVHcb8zf/o4HrME4fOGw1KUQEfUbl0HQ3NwMAEhOTkZYWBhSUlJgMBgc2hgMBqSnp0OpVCIj\nIwMVFRX2cQ899BACAwM7zNdoNGLlypXw8/PDihUrOszTYzz+ONKqvLGnco/UlRAR9RuXQVBaWgqV\nSmUfVqvVOH78uEMbo9EItVptHw4ODkZ1tesDrHfPV6VSwWj00P3wixYh7ZNvsbfiA7Rb26Wuhoio\nX/T5MtSiKHY4eUEQhC6n6a6cnBz7a61WC61W25Py+mboUEzWLsFE82GUXC5Bcljy/ftsIqJuKCoq\nQlFRUZ/m4TII4uPjsX79evtweXk5FixY4NBGo9Hg3LlzmD9/PgCgrq4O4eHhLj80Pj4eFRUViImJ\nQUVFBeLj4ztte3cQSOInP0HaHw9jT+UeBgEReZx7/0B+9dVXezwPl7uGAgICANh6DtXU1KCwsBAa\njcahjUajwe7du9HQ0AC9Xo+oqKguP1Sj0SA/Px+tra3Iz89HYmJijwu/bx59FGn/YxPer3ift64k\nokGpy15DW7ZsQWZmJh577DGsWrUKQUFB0Ol00Ol0AICEhAQkJSUhLi4Omzdvxuuvv26fNiMjA3Pm\nzMEXX3yBSZMm4e233wYAZGVl4dKlS5g2bRq+/vprPPvss/20eG7g7Y0Zj/0Evl6+KPu2TOpqiIjc\njhed66ZfFP4Cft5++PdH/l3qUoiIOsWLzvWjNFUa9lSwGykRDT4Mgm7ShGhwpfUKzjecl7oUIiK3\nYhB0k0JQIHXCPOypeF/qUoiI3IpB0ANpO0uwp/T/Sl0GEZFbMQh6QDtvBT5vqsI3pm+kLoWIyG0Y\nBD3gm7EMC88DH3z2ntSlEBG5DYOgJ6ZMQVrrZOw5/rbUlRARuQ2DoIcWPPovMDR+hsbWRqlLISJy\nCwZBDw3/0TLM85qKfV/sk7oUIiK3YBD01JgxSEv9n7xHARENGgyCXnhi2hM4fPEwrpuvS10KEVGf\nMQh6QTlUibgJcThYfVDqUoiI+oxB0EtpqjTuHiKiQYFB0EtLVEuw74t9MLebpS6FiKhPGAS9FDJi\nIiK+M+PICZ5cRkQDG4OgtwQBadZp2HP4P6WuhIioTxgEffDko8/jg6ulsIpWqUshIuo1BkEfTPv+\nMgS0iSj99M9Sl0JE1GsMgr7w8kLa0BjsKcyTuhIiol5jEPRR2rznsKfttMfcW5mIqKcYBH0U++gz\naB2rREV9hdSlEBH1CoOgjwSFAktUS/A+b2FJRAMUg8ANeJYxEQ1kDAI3eCjsIXzZ9CW+bPpS6lKI\niHqMQeAG3gpvPDHtCXxQ+YHUpRAR9RiDwE2eVKVhz8E3gRs3pC6FiKhHGARu8nhECk7fvIS6j/4k\ndSlERD3CIHCTId5D8PiIWfjoIK89REQDC4PAjdKSM7Gn7TRgMkldChFRt3UZBMXFxYiKikJkZCTy\n8pxfSmHDhg0IDw9HbGwsKisru5w2JycHISEhiImJQUxMDPbv3++GRZHeopilODIZML3/jtSlEBF1\nn9iF6Oho8ciRI2JNTY04bdo0sa6uzmG8wWAQ586dKzY0NIh6vV5ctGhRp9PW19eLoiiKOTk54ubN\nm7v6aLEb5Xmc+b99QHz3J7OkLoOIZKo3602XWwTNzc0AgOTkZISFhSElJQUGg8GhjcFgQHp6OpRK\nJTIyMlBRUdHptMePH787gNwYZ54jbe4/Y8+jE6Uug4io21wGQWlpKVQqlX1YrVY7rMwBwGg0Qq1W\n24eDg4NRXV3d5bR5eXlITEzEpk2bYBpE+9RTH/gh9td9ihsWdiMlooHBu68zEEWxw1/3giC4nCYr\nKwuvvPIKrl69ivXr10On0yE7O9tp25ycHPtrrVYLrVbb15L71Tj/cVAHq/HXi3/F9yO/L3U5RDTI\nFRUVoaioqE/zEEQX+2iam5uh1WpRVlYGAFi9ejUWLFiARYsW2dvk5eXBYrHgxRdfBABERESguroa\nTU1NeOSRR1xOCwBnzpzBqlWrUFJS0rE4QRiQu5A2f7oZnzd8jt8/8XupSyEimenNetPlrqGAgAAA\ntt4/NTU1KCwshEajcWij0Wiwe/duNDQ0QK/XIyoqCgAwatSoTqetra0FAFgsFuj1eixcuLBHRXu6\ntKg0fPj5h2i3tktdChFRl7rcNbRlyxZkZmbCbDZjzZo1CAoKgk6nAwBkZmYiISEBSUlJiIuLg1Kp\nREFBgctpAeCll17C6dOn4evri+TkZGRlZfXT4kkjPDAc4/zH4W9H9Uh6+BmpyyEicsnlriGpDdRd\nQwCQ88lGXN32Bt7Y+Ckwc6bU5RCRTLh91xD1XlrUk9gz3QviO3qpSyEicolB0E8eGPsABH9/nNn/\nf4ABulVDRPLAIOgngiAg7YGl2DPVDOzh3cuIyHMxCPpRWlQa9mgCgGefBe45I5uIyFMwCPrRgyEP\n4jvRhIq//BGIjZW6HCIipxgE/chL4YVXkl9BytGfofS7MqnLISJyit1H74MPKz/Ezz76GXLn5+Lp\nB56WuhwiGsR6s95kENwnZ787i9Q/peJH03+EX837FbyuXQeGDwcU3CgjIvdhEHi4+uv1+OGffwh/\nX3/sKhmHkaabwM6dgHefr/1HRASAJ5R5vKBhQTi47CAmjZyERNUxVDVfBH74Q+AGL1lNRNJhENxn\nPl4+eGvRW1itWYOkpC9wOOAK8IMfAC0tUpdGRDLFXUMS+uTiJ8jYnYGXL0/Bc0YRwsFCYMQIqcsi\nogGMxwgGoAuNF5D6p1Q8eGMM/vO5/w9fnyFSl0REAxiPEQxA4YHh+HTFp/hunD8eK0hB3bU6qUsi\nIplhEHiAEX4jsGfpHiSHJSP+f8fjzLdnpC6JiGSEu4Y8zJ8++xNWf7wauh/o8KQqDeji/s9ERHfj\nMYJB4uQ3J5H2bhr++awP/tdP8yE8/LDUJRHRAMEgGES+bfkWab+fh0mnqvH20ncw/IknpS6JiAYA\nBsEg02Zpw7NvP4VTZw9ife0ULHlwOUb8ZDkwbpzUpRGRh2IQDEKiKOL9s+/iD5/k4uiV0/j+hGQ8\n/chapESkwNfLV+ryiMjDMAgGufrr9fhz+Z+x6+wufN7wOdLV6Xh65tOYM2kOFOergMhIHlwmkjkG\ngYzUNNXgnbPvYNfZXWi5YUJGyVU8fdEfMx7NAJ58EkhI4JVNiWSIQSBDoiji79/9HbvO7sI7p/4I\nZauIp/8OZJQLmPRPq4F/+zepSySi+4hBIHNW0YriL4uhP6vH7s/+H2YMCcXTyc8jXZ0O5VCl1OUR\n0X3AICC7G5Yb+LjqY+w6uwsHqw9izqQ5iJ8Qj9jxsZj9l1MI+csxCIkPAomJtkdQkNQlE5EbMAjI\nqea2Zhy+eBinak/hVO0pnPz6BKzmG5htDkLsV1bMPv0dZrePwZRfvQVh4UKpyyWiPmAQULeIooja\nllqc/ObkrXA4iVOXS9HS3orZE2Mxe/xszB43G7PHz0bk6EgoTp4CJkywPYjIozEIqE/+ce0f9q2G\n24/66/WIbvHH7HONCG20wsdnCHz8hsLHbyi8/yULPhNC4KPwgY+XD3wUPvBWeMPn2KfwMVvh4z8S\n3v4j4eM/Ej4jRsEnJAw+Q4bZ2tw1jY+XbTovwQuCJ3R/NZmApibbo7n5znNqKuDv37H9hx/auu0G\nBDg+Ro1yT8+tlhbb49o1x0dyMuDr5FySy5dttz/18wOGDLE9e3n1vQ7qHw0Ntn9fk8l2H/MpU/o0\nu34JguLiYmRmZsJisWDNmjVYvXp1hzYbNmzAu+++i8DAQOzatQsqlcrltCaTCcuWLUNZWRlmz56N\ngoIC+Dv5D8YgcJ+ioiJotdoeT3el9QpOf3saJ785gdorl2Buuw7zDdvDEjACZkGE2WqGud0Mi9Vi\ne131Bcxt1+68J1pgFtthnjQBZgXutGs3255br8EsiLAqAJ92wMcKeFthCw/fIQ5h4a3whnjpEqwW\nM9oFwKqA7VkArMpAtCtsB83bre2wilbb67ZW27MgwgoRImyX3fXy9oVC4QWFoIDX7WfBC4orjVC0\nW+ElKKAQvGzPCi8ogsfAy8cXCkGB1vOtGP694bYv6ZtvgJtmwNoOtLcD7Vbb68hIwMen45f6zTeA\nKNoeVuudR2gY4O1khf3557bxCoXtIdx6Dgt1voKvqgLMFkC0AtZbnyMAUEU5n39Nja2dQrg171vP\nE8Y7D7LGJtvz7XYAANF2UyVnQd5wxVa/KNra3V72MWNt8wBw7Ytrd77P+oa75n9XTZ3N32KxvS/e\nquP2s7ePbbnv1dZ2q4Z72g8b7ry9yeSkPYCAkc7rqauz/Q5u/7u233oOC7Mvr4PKSgCC7bsOGIm/\nvHgSkwImOSmke/olCGJiYvDmm28iLCwM8+fPx7FjxxB014FFo9GIdevWYe/evThw4AB27dqFffv2\nOZ22pKQEo0ePxm9/+1tcvnwZv/vd7/Dzn/8ckydPRnZ2tlsWiJzLyclBTk6O1GU4J4pAezus5puw\nmG/AbG6D+WYbzMOGwKwQbcFxKzQsVgsUly5D0dpmW1m3W23PFisUs2OhGO5vW5nftXJXfLgXXlea\noGhvh5fFCgCwjgqAddFCWANG2kOjXbwrPLp4763X30JWdlbvlvfwYaCx0bYSHzoUGDbU9jwr2vlf\n+H0liraVpbe38xXX2b8Dra3AjZvAzRu2e2jfuAksWuS8nk2/sW0h3bgJ3GizveflDfz617a/aO+V\nt9UWTN6NF+VzAAAFFklEQVRethq8vG3Py54G/Gw3Ytr+u+13vs+3tgGmllu13Lzz/Jvf2L6ne/1g\nkS2cFAKg8AK8FLbnDz8AhjvZgvvpPwGtbXfa3X5+6y1g2LCO7X+xHrh5804Ie916fjUHGOKkHp3O\n9m87fDgwfJhtnsOG287t8fbu2P4e3xv9Pfh5+3XZrjO9Wm+KLjQ1NYnR0dH24dWrV4v79u1zaLN1\n61YxNzfXPhweHt7ltE899ZRYVlYmiqIonjx5UkxPT3f6+V2URz2wceNGqUsYVPh9uhe/T/fpzXrT\n5Q7M0tJS+24eAFCr1Th+/LhDG6PRCLVabR8ODg5GdXW1y2nvHqdSqWA0GnuWXkRE5DZdb6d0QRTF\nDpshnR3wu/3+ve1d8YiDh4PEq6++KnUJgwq/T/fi9ykdl0EQHx+P9evX24fLy8uxYMEChzYajQbn\nzp3D/PnzAQB1dXUIDw+HUqnsdNr4+HhUVFQgJiYGFRUViI+Pd/r5PQkMIiLqHZe7hgICAgDYev/U\n1NSgsLAQGo3GoY1Go8Hu3bvR0NAAvV6PqKgoAMCoUaM6nVaj0SA/Px+tra3Iz89HYmKi2xeMiIi6\np8tdQ1u2bEFmZibMZjPWrFmDoKAg6HQ6AEBmZiYSEhKQlJSEuLg4KJVKFBQUuJwWALKysrBs2TJM\nmzYNs2fPxqZNm/pp8YiIqEtuPmDtFkeOHBFVKpU4depUcevWrVKXM6CFhYWJM2fOFKOjo8X4+Hip\nyxlwli9fLo4ZM0acMWOG/b2rV6+KixcvFidNmiSmpqaKJpNJwgoHFmff58aNG8WJEyeK0dHRYnR0\ntPjxxx9LWOHAcenSJVGr1YpqtVp8+OGHxV27domi2Lvfp0desH7t2rXQ6XQ4dOgQtm3bhvr6eqlL\nGrAEQUBRURHKysrYO6sXli9fjv379zu8t337doSGhuL8+fMICQnBjh07JKpu4HH2fQqCgHXr1qGs\nrAxlZWUdjkOScz4+PsjNzUV5eTnee+89vPzyyzCZTL36fXpcEDQ3NwMAkpOTERYWhpSUFBgMBomr\nGthEHnTvtYceegiBgYEO7xmNRqxcuRJ+fn5YsWIFf5894Oz7BPgb7Y1x48YhOjoaABAUFITp06ej\ntLS0V79PjwuC7py7QN0nCALmzZuHJUuWYO/evVKXMyjwPBj3y8vLQ2JiIjZt2gSTySR1OQNOVVUV\nysvLkZCQ0Kvfp8cFAblXSUkJzpw5g9deew3r1q3Dt99+K3VJAx7/enWvrKwsXLx4EQcOHEB1dbW9\nMwp1j8lkwtKlS5Gbmwt/f/9e/T49Lgji4+NRWVlpHy4vL2f30j4YP348ACAqKgqLFy/GRx99JHFF\nA9/t82AAuDwPhrpnzJgxEAQBAQEBeO6557Bnzx6pSxowzGYznnrqKTzzzDNITU0F0Lvfp8cFQXfO\nXaDuuX79un0zu66uDgcOHOCBODfgeTDuVVtbCwCwWCzQ6/VYyJsjdYsoili5ciVmzJiBF154wf5+\nr36f/di7qdeKiopElUolRkREiG+++abU5QxYFy5cEGfNmiXOmjVLnDdvnrhz506pSxpwfvzjH4vj\nx48XfX19xZCQEDE/P5/dR/vg9vfp4+MjhoSEiDt37hSfeeYZcebMmWJsbKz44osvig0NDVKXOSAc\nPXpUFARBnDVrlkPX2978Pj36xjRERNT/PG7XEBER3V8MAiIimWMQEBHJHIOAiEjmGARERDLHICAi\nkrn/BlCP9fE4WD5FAAAAAElFTkSuQmCC\n"
      }
     ], 
     "prompt_number": 253
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "h,b = histogram(array(afli)*num_haplo,bins=range(0,num_haplo+1))", 
      "plot(array(h,dtype=float)/sum(h),'--r')", 
      "plot(sfs,'g')", 
      "", 
      "ax = figure(1).axes[0]", 
      "ax.set_yscale('log')"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "display_data", 
       "png": "iVBORw0KGgoAAAANSUhEUgAAAXwAAAEBCAYAAAB7Wx7VAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAIABJREFUeJzt3XtYVHXix/H3KHgLQ1EETQQvbECBQiGuJuIldCUvZWZW\nZGKbpqnZPtb22/1tWLlmV821slbNddOy0rK8YoaY5aWgcvlpJjqbppaKICko4vn9MUmaVjPDDGcu\nn9fzzGMcmZmP80yf+c73fM85FsMwDERExOfVMTuAiIjUDhW+iIifUOGLiPgJFb6IiJ9Q4YuI+AkV\nvoiIn1Dhi4j4CRW+iIifCHDng586dYqHH36Y8vJyBg0aRL9+/dz5dCIi8ivcOsLftGkTycnJvPji\niyxdutSdTyUiIr/B4cLPysoiLCyM+Pj4C7bn5eURGxtLdHQ0s2bNAmD79u20b98egPLychfEFRER\nZzlc+CNHjmT16tUXbZ84cSJz5sxh3bp1zJ49myNHjpCQkMCePXsAaNSoUc3TioiI0xwu/O7du9O0\nadMLtpWWlgKQmppKZGQk6enpbNmyha5du/Lpp58yfvx4brrpJtckFhERp7hkp+22bduIiYmp/jku\nLo7NmzeTkZHB008/7YqnEBGRGnLrKh17WSwWsyOIiHglR85w75JVOsnJyezcubP658LCQrp06eLQ\nYxiGoZuLbo888ojpGXzpptdTr6Wn3hzlksIPDg4GbCt1rFYrOTk5pKSkOPQYf/nzZHJzc10RR0TE\np+Xm5pKdne3w/Rwu/OHDh9O1a1d27dpFREQE8+fPB2DGjBmMHj2aPn36MHbsWJo3b+7Q497QM4a0\ntDRH44iI+J20tDSnCt/hOfzFixdfcnuPHj3YsWOHwwHOKdixnt/3HeX0/eUn+uB0Lb2erqPX0lwW\nw5mJIFeHsFhIvC6UZx9bojeEiMhvyM3NJTc3lylTpjg0l+85hT+1Dfn/81+zo4iIeA2LxeJQ4XvM\n2TJ3Goc5XXXa7BgiIj7LYwo/6OMgFryzwOwYIiIez9lVOh4zpZO5NJMekT0YlaQdtyIi9vDaKZ2k\nlkkUHCowO4aIiM/ymMJPDE8k/2C+2TFERHyWR5xLB2D1P5ZRcHQbVSOrqFunrtlxREQ81rllmY7y\nmDl84+RJov98Gcsf+pzYVglmRxIR8XheO4dPw4Yk/hBEfsFKs5OIiPgkzyl8IKl+FPm788yOISLi\nkzxmDj87O5vLKlpRcLTQ7CgiIh7N++fwDYPDby8k+vMsjj16WhdFERH5Dd47hw+EpvajccNg9pbs\nNTuKiIjP8ajCJzSUpHbdKDioA7BERFzNswqfHw/AOqQDsEREXM3jCj+pZZKOuBURcQOPWqWTlpZG\nUpKt8A3D0I5bEZFL8IlVOgCGYdDi6RZ8MeYLWjVuZXIyERHP5dWrdAAsZ86QdNBC/refmh1FRMSn\neFzhExhIkvUU+TvWm51ERMSneF7hA4lBHSiwbjY7hoiIT/HIwk9qnUz+8Z1mxxAR8SkeWfjt4lMp\nqTrB0ZNHzY4iIuIzPKbws7Ozq5cZ1enYiU5HAnXJQxGRS/D6i5hfEOPMGSYtvouW7RJ4sNuD5gUT\nEfFgXr8sE4CAAJIS+uqIWxERF/LMwgcSWyZqSkdExIU8tvBjmsew//h+jp86bnYUERGf4LGFH1An\ngPgW8Xxx6Auzo4iI+ASPLXzQtI6IiCt5dOEnvfw++UUfmR1DRMQneHbhE07+/m1mxxAR8QkeXfhX\nt+vC7vJvqThTYXYUERGv5zGFf/6RtufUT0jid6eC2P7ddnNCiYh4IN860vacTz8la3Y6KeOmMfra\n0bUfTETEg/nGkbbnxMWRuPsEBQc+MzuJiIjX8+zCb9SIpPmryf/uc7OTiIh4Pc8ufKBjm2QKDxdS\nWVVpdhQREa/m8YUfVC+IiMsj2HlEF0QREakJjy98gKSWSTpzpohIDXlP4R9S4YuI1IRXFH5iUAcK\nDqjwRURqwjsK/76pfH4gn7PGWbOjiIh4La8o/JC4a2hmNGB38W6zo4iIeC23Fv7evXu5++67GTp0\naM0eKD6exBOXU3BQp0oWEXGWWwu/bdu2/POf/6z5AyUkkLTvjHbciojUgF2Fn5WVRVhYGPHx8Rds\nz8vLIzY2lujoaGbNmuWWgADEx5O0/Qj52nErIuI0uwp/5MiRrF69+qLtEydOZM6cOaxbt47Zs2dz\n5MgRFi5cyKRJkzhw4IDrUjZpQmJ4JwoO5jt0oiAREfmJXYXfvXt3mjZtesG20tJSAFJTU4mMjCQ9\nPZ0tW7aQmZnJc889R6tWrSguLmbMmDF8/vnnTJ8+vUZBW679hMCAeuw7vq9GjyMi4q8CnL3jtm3b\niImJqf45Li6OzZs3k5GRUb0tJCSEl156ya7HO//czmlpaaSlpV30O+eOuG0T3MbZ2CIiXis3N/ei\n64Y4wunCdzV7TuafGG67qPngmMHuDyQi4mF+PhieMmWKQ/d3epVOcnIyO3f+dEKzwsJCunTp4uzD\n2UXn1BERcZ7ThR8cHAzYVupYrVZycnJISUlxOsilLnH4cyp8ERE3X+Jw+PDhbNiwgaNHj9KiRQse\nffRRRo4cyYYNGxgzZgyVlZVMmDCBCRMmOJPd7st0GQUFNFvTix337SQsKMyp5xIR8RWOXuLQs69p\n+3MjRtA7diuTb36Ofh36uT+YiIgH89pr2tozpUN8PEkljTStIyJ+za1TOu5m96fU2rUsmjeJZUPi\neHPom+4PJiLiwbx2hG+XhASSPjugEb6IiBM8pvDtmtIJCyO6tC7flR2ipKKkVnKJiHga/5jSAfjb\n3+gWtoKpGc+QFpXm1lwiIp7Mt6d0AB59lKR2XTWtIyLiIO8rfGynWFDhi4g4xmMK3645/B8ltUyi\n4JCufiUi/sl/5vCB01WnafJEEw5PPsxl9S5zYzIREc/l+3P4QL269YgNjeXL7740O4qIiNfwysJn\n3TqSAtpoWkdExAEedT78X7rwyUU+/pik0+V81lw7bkXE/zh7IRSvnMPnnXfYvPgpxvYqJ3+0Sl9E\n/JNfzOGTkEDC1v+y88hOTledNjuNiIhX8M7Cj4qi0ZFS2l0eSeH3hWanERHxCt5Z+HXqwNVXkxjY\nRgdgiYjYyWN22jrswQdJqrtNK3VEROzkMSN8R460BeDGG0lK6KsRvoj4Hb860vac0opSrnj2Ckr/\nXErdOnXdkExExHP5xyqdHwU3CCY8KJxdR3eZHUVExON5deGD7URqmtYREfltXl/4ieGJ5B9S4YuI\n/BbvLvz33ydpxzEKDmqljojIb/Huwi8rI3H9DvIP5ju101dExJ94TOE7vCwTID6eFgW7CKoXxN6S\nvW7JJSLiafxyWSaVlXD55Qx8pReZiXcx9Kqhrg8nIuKh/GpZJoGBcOWV3HjZtUzdOJWKMxVmJxIR\n8VjeXfgA8fHcVdyG6GbRPLDmAbPTiIh4LO+e0gHYsQMaN6a0eWOSXk5iWu9p3HLVLa4NKCLigRzt\nTu8v/PN8euBT+r/Wn09GfUL7kPYuSCYi4rn8aw7/Z65tdS1/Tf0rw94axqkzp8yOIyLiUXxqhA9g\nGAZDlgwhIjiCmf1muuQxRUQ8kV+P8MH2AswdOJflXy1n2Y5lZscREfEYHlP4Th14db7du+G07fq2\nTRs25fUhrzP6/dFYS6wuySci4in888Cr891zD+zfD2+/DQ0bAvDsJ8/yRuEbbBy5kXp167kgqYiI\n5/DfKZ0XXoCQEOjfH8rKAJjUZRJhl4Xx8AcPmxxORMR8vlP4AQGwYAFER0N6Ohw7hsVi4dXBr/LW\n/73Fe1+9Z3ZCERFT+U7hA9StC3PmQEoK9O4Np08T0jCExUMWc/d7d/NN6TdmJxQRMY3vzOGfzzBg\nyxbo0qV605ObnuSdne+w4a4NBNYNdN1ziYiYxK+PtP01Z42z3LDoBuLD4pneZ7pbn0tEpDb4707b\n31DHUocFgxewaPsiVn690uw4IiK1zm8KHyC0flNeu+k1st7NYv/x/WbHERGpVf5T+D/8AAkJpB65\njPGdxzP87eGcOXvG7FQiIrXGfwo/KAimTYM//IGH66TSMKAhj+Q+YnYqEZFaE+DuJ3j33XdZsWIF\nZ86cYcyYMXTu3NndT/nLBg2CBg2oc9MQFi54gaTPJ9Ijsgfp7dPNyyQiUktqbZXO999/zyOPPMKL\nL754cYhaWKVzgbw8uPlmPpw5idsOPM9n93xGq8atau/5RURcwG2rdLKysggLCyM+Pv6C7Xl5ecTG\nxhIdHc2sWbN+8f7Tp09n9OjRdgdzq9RUeP99elZFMOaaMdy+9HaqzlaZnUpExK3sHuFv3LiRoKAg\n7rzzTrZv3169PTExkZkzZxIZGUnfvn356KOPWLVqFfn5+UyePJmWLVvy0EMP0bdvX3r37n3pELU9\nwj9P1dkq0v+dznVtrmNK2hRTMoiIOMPR7rR7Dr979+5YrdYLtpWWlgKQmpoKQHp6Olu2bCEzM5PM\nzEwAnn/+edavX09ZWRm7d+/2nFH+j+rWqctrN71G0pwk0tul061NN7MjiYi4RY122m7bto2YmJjq\nn+Pi4ti8eTMZGRnV2yZMmMCECRNq8jRuFx4UzvN/eJ4/vvdHCkYXUD+gvtmRRERczu2rdOx1/sn8\n09LSSEtLq9XnHxI7hH/lzuTJVX/lfwc8VavPLSJij9zc3BpdKMqhVTpWq5UBAwZUz+GXlpaSlpZG\nQUEBAOPHj6dfv34XjPDtCmHiHP759j02maSKmXw08QuubBFrdhwRkV9Vq+fSCQ4OBmwrdaxWKzk5\nOaSkpNTkIU0V8T9P8Lc9Edwz5wbOGmfNjiMi4lJ2F/7w4cPp2rUru3btIiIigvnz5wMwY8YMRo8e\nTZ8+fRg7dizNmzd3KkiNr2nrCnXrMnbqWioOfsO8d7PNzSIi8gt0TVsX+nLeNPp8/b98+cDXhIe2\nNTuOiMgl6Xz4LvLwQ9eyt0MzXv/jGrOjiIhckteeD98jpnTO87fHN/JpeRErdq0wO4qIyAU0peMG\n6/asY9TyURSOLSSoXpDZcURELqApHRcb8c4ImjVsxrN9nzU7iojIBTSl42LPpD/Dou2L+PTAp2ZH\nEREBNKXjVgu/WMiz7z3M1gHLCeyYZHYcERHAi0f4nuyOhDtoHhjMjEf7Q0WF2XFERJyiEb6dio7u\nJmXGVWw9cRvtnp1vdhwREe8d4XvqHP457Zt14MHUh7m3bDHGqlVmxxERP6Y5/FpQWVVJ8nOxTF5+\nlNvf3gWhoWZHEhE/5rUjfG8QWDeQV4Yv5k/Xn+Vo6SGz44iIOEQjfCfcv/p+Sk+VMn+Q5vJFxDxe\nO8L39Dn88z3W8zHW713P+r3rzY4iIn5Ic/i17L2v3uOBtQ/w5ZgvaRjY0Ow4IuKHvHaE720GXDmA\nxPBEHst7zOwoIiJ20Qi/Bg6WHSThpQQ+KLqOhFvGQ69eZkcSET+iEX4tatm4JVN7TeWe6J1U3XUn\nFBebHUlE5Bd5TOF7007b892ddDeBIc15cXg0/PGP4IXfVETEu2inrYl2HN5B9/ndKVgZQUS3/jB1\nqtmRRMQPaErHBLGhsdzX+T7G3tWCs28ugeXLzY4kInIRjfBd5NSZU/T+V286No7mH4PmYKlXz+xI\nIuLjNMI3Sf2A+qy4bQVbS/7DQxv+6vUfYCLie1T4LhTcIJjVt69m1e5VWp8vIh5Hhe9izRo1Iycz\nh9e2v8YzHz9jdhwRkWoBZgc4Jzs7m7S0NNLS0syOUmPhQeGsy1xH6qupNCKAexf8Hzz1FFx+udnR\nRMQH5ObmOrWMXTtt3aiouIi0BWlM/SaaOwsD4P33QTtzRcRFtNPWg7QPac/aO9byUOsdvNnyGNx9\ntw7MEhHTqPDdLDY0ltW3r+a+q79hRclW+MtfzI4kIn5KhV8LOoZ3ZPnw5dzV7QgffLQQdE1cETGB\nCr+WpLRO4a1hb3NrRjmb4oLMjiMifkiFX4t6RPXg30Ne48YlQ/jswGdmxxERP6PCr2V9O/Tl5QEv\nk7Eog/98/x+z44iIH/GYdfj+ZHDMYMory+n77758OOJDftfsd2ZHEhE/oMI3yfD44ZysPMn1C/qw\n4T/XEjXjVR2YJSJu5TGF70tH2tprVNIoTpz+gT7H/kberRm0eucDHZglIr9JR9p6sWl5U1m48gk2\nHOxL6PwlUEe7VkTkt+lIWy/0cOpfuKnnWNKbr+LYxHugstLsSCLig1T4HuKx9CdIS72TPg3e5NA/\nZ5gdR0R8kArfQ1gsFp4d+AID+0+i6+kX+erIV2ZHEhEfozl8DzQ3fy5/Wf8Xlg1bxu8jfm92HBHx\nUI52pwrfQ638eiUj3hnBKwNeYXDMYDh7VjtzReQC2mnrI/pH92fV7asYu2IsL3zyPCQlwcqVZscS\nES+mEb6H23NsD/3+3Y8hjTvz9//NxXL7HfDYYxAYaHY0ETGZpnR80JGTR7hh0Q1EX9aGuQtLqXf8\nBCxeDBERZkcTERNpSscHNW/UnPUj1nPccoqMW89yPKMPpKRASYnZ0UTEi7i18Hfu3Mm9997LqFGj\nWLp0qTufyuc1CmzE27e8TYdmHejedBkHPlwOTZqYHUtEvEitTOmcPn2aESNGsHjx4kuH0JSO3QzD\n4ImPnuClz15i1e2riAuNMzuSiJjELVM6WVlZhIWFER8ff8H2vLw8YmNjiY6OZtasWZe87/Lly+nZ\nsye33HKL3aHkl1ksFh7u/jCP93ycngt6svG/G82OJCJewq4R/saNGwkKCuLOO+9k+/bt1dsTExOZ\nOXMmkZGR9O3bl48++ohVq1aRn5/P5MmTadWqVfXvDhw4kOXLl186hEb4TskpyuH2pbfzQsYL3Bx3\nM+TmwurVWsUj4icc7U67To/cvXt3rFbrBdtKS0sBSE1NBSA9PZ0tW7aQmZlJZmYmABs2bGDp0qUY\nhsHQoUPtDiX2ub799azNXMsNi25g//H93H/V7TBtGtxwA7z9NgTp2rki8hOnz4e/bds2YmJiqn+O\ni4tj8+bNZGRkVG/r0aMHPXr0qFlC+VWdwjuxKWsT/V7rx77j+3jq/feoM3Yc9OwJK1ZAixZmRxQR\nD+FRF0A5x98uhFJTkU0i2ZS1iUGvD2L4u5kseOFVGjw2Dbp1s03xtG9vdkQRcQFnL3xyjt2rdKxW\nKwMGDKiewy8tLSUtLY2CggIAxo8fT79+/S4Y4dsdQnP4LlFxpoI7lt7B0fKjvHvru1w+fxHExIA+\nPEV8Uq0deBUcHAzYVupYrVZycnJISUlx9uHEBRoENOCNm98gtnksaa+m8d0dN6rsRaSaXYU/fPhw\nunbtyq5du4iIiGD+/PkAzJgxg9GjR9OnTx/Gjh1L8+bNnQ6SnZ1do68qYlO3Tl1m95/NoCsH0W1e\nN4qKi8yOJCIulpube8E0uL10Lh0f9tKnL/HohkdZcdsKElsmmh1HRFzMa8+loxG+6425dgzP/+F5\n+v67L7nWXNvG1avhT3+ynV9fRLySRvjyiz7c+yHD3hrGixkvMqRlLxg40HamzfnzoX59s+OJiJO8\ndoQv7tOzbU/W3LGG8avGM6doCaxdCxUVkJEBx4+bHU9EaokK308ktkwkb2QeT378JI9ufQpjyRKI\njoYePeDQIbPjiUgt8JjC1xy++3UI6cCmrE0s27mM8Wvvp+ofs+CWW2DfPrOjiYgDNIcvdiutKGXw\nG4MJbRTKwhsXUj9A8/gi3khz+PKbghsEs+r2VVQZVWQsyqDsVJnZkUSkFnhM4WtKp3Y1CGjAkpuX\n0CGkA2kL0vj+xPc//aW+bYl4NE3piFMMwyB7QzaLti9i7R1radu0re3UylOnwrBhtltUlNkxReQS\n3HI+fPFdFouFKWlTaNGoBd3nd2fFbSvoOHgwNGsGr78Oycm2s23eeqvtFh5udmQRcZJG+FJtSeES\n7lt5H28OfZMeUT9ex6CyEj74AN54A26+2bZ2X0Q8gqPdqcKXC3yw5wOGvTWMNsFtiGoSRdumbYkK\njiKqyU+3xvUbX3in8nJo2NCcwK527pQTdX62e6uoCN5/H+66C348U6yI2bx2Sic7O1sXPvEAvdv1\npmhCEV8Xf83eY3uxlljZeXQnq3avwlpixVpipVFgo+ryb9u4DVFPv0JUy1iieg0havAILgvxwmmf\nH36ABQtg5kx46SXo1evCvz97FjZvhuxs29TWuHFw9dW1n3P/fvj+e4iP13WL/ZizF0LRCF8cYhgG\nh08erv4wsJZYsR7exd6vt2Et3sN/A07Q2FKfqMvb0L7dNXQI6UD7pu2r/wwPCsdisZj9z/jJvn3w\nj3/A3LmQmgqTJsF118EvZTx4EF5+2XaLjrZ9QHTs6Lo8X30Fy5bBrl0wb97Ff79yJTz0EOzZAwkJ\n0Lmz7dajB7Ru7boc4hU0pSOmOnv0CN+//S/2HN1NUf/fU3SsiKJjRewu3k1RcREnTpXR/mQD2tcN\npf1lrenQtD3tw+No/7sU2lzZmYA69n3pPHP2DMfKj1FcXnzxraKYoyePUlxeTGDdQEIahtCsYTNC\nGoZcePv8K0KyxnH58LuoM34CtGtn/z+0stJWzF26QJs2Tr5a2JbAfv45LF1qu5WUwODBcNNN0Lv3\nL9+vrAzy82HrVtutb1+4+27nc4h7GAZs22b78C4thcWLXfrwKnzxaMf37qQoZwlFh7+iqNTK7lMH\nKDKKKWpi8F2900QER1R/IwhuEMyxQ1aKv7NSXP8sRy0VFJ8uobi8mBOnT9CkQRNbmTf6WZk3sP3Z\ntGFTzpw9c+kPhfJijp48QnH5MU5U/vRYP781bdiUAIujM5/n3sv2fJMx4NVXoVUriIuDK1r/8rcL\nZ6xdC8XFttF/VCS0uuLi/RPieidPwhdfQP5ncLoSrkmCTolw+eXVvxIRHMFdne6q0dOo8MVrnTpz\nir0leykqtn0rKK0opdnBEkLW5BHyzWFC9hwkpF4wIRG/s43KR9k5oi0pse1U/oVTQVdWVVJSUXLJ\nD4VjFceoMqoc+4d8e8A29ZKcDFddBYEBUFVlu9Wr59hj1VRJiW3ef/+38F+r7eyokVFw/fXQtEnt\nZvEXhgGzX4DWV9hKPrLNJT/EI4MjuTupZt/KvLbwH3nkEe20lV939qytvL7+Gpo0gWuuufh33noL\n1q2zza+3awfr18Nrr9kOJuvZs/Zyrllj2zewdSt06wZ5efDss7ZVPmY6dMi2zDYjw/Ya1qYPP7S9\nDr1726bCAjxmzYjrVVa6daf6uZ22U6ZM8c7C94AY4gsKCyE31/ahsHu3bcfmuHFwxRXm5Ckqgo8/\ntpVcq1bmZLBXebntovepqbZvANddB40a2XffU6dsr/mOHRAaanucn8vLgxUrbNNM33wD6enQvz/c\ncAM0berKf4n7lZfb9uE0aWL7N5jEa0f4HhBDxL9VVcGWLZCTY/uWVFBgWwF0001w330X//7mzfD3\nv9tKft8+2yk4YmNh6FC47bZff65vv7VdbnPFCpg40bbKyGynTtmK/FLffD75BKZPty2JPXzY9k2p\na1d48MFf37nuZip8EXGNsjLYsAG++w5Gjbr4761W20qh2Fjb6TdcvX9i3TrbfhB3HOj22We2YyoO\nH/7pVlFhO8biX/+6+Pe/+cZ2n9BQ2y083CMOwFPhi4j3q6qCQYNsHzjXXGPb59C/v20l06VWMX33\nnW3/zf79ttu+fbY/r7zS9i3i5w4etH2bCQ2FFi1sfwYHu3aFVC1Q4YuI7zh50razd+VKW3FfcQVs\n2nTx7xUVwdNPQ0SEbQlq69a2/77iCvv3Q3ghFb6I+CbDsM2hh4WZncRjeO0Vr3QBFBH5VRaLyv5H\nugCKiIif8doRvoiIuJcKX0TET6jwRUT8hApfRMRPqPBFRPyECl9ExE+o8EVE/IQKX0TET3hM4etI\nWxER++hIWxERP6MjbUVE5JJU+CIifkKFLyLiJ1T4IiJ+QoUvIuInVPgiIn5ChS8i4idU+CIifkKF\nLyLiJ9xe+CdOnCA5OZkVK1a4+6lERORXuL3wn3zySYYNG+bupxERkd9gV+FnZWURFhZGfHz8Bdvz\n8vKIjY0lOjqaWbNmXXS/nJwc4uLiCA0NdU1asYtOQudaej1dR6+luewq/JEjR7J69eqLtk+cOJE5\nc+awbt06Zs+ezZEjR1i4cCGTJk3iwIEDbNiwgc2bN7No0SJeeeUVnSCtluh/KtfS6+k6ei3NFWDP\nL3Xv3h2r1XrBttLSUgBSU1MBSE9PZ8uWLWRmZpKZmQnA448/DsCCBQsIDQ3FYrG4KreIiDjI6Tn8\nbdu2ERMTU/1zXFwcmzdvvuTvjhgxgv79+zv7VCIi4gJ2jfBrg0b/rjVlyhSzI/gUvZ6uo9fSPE4X\nfnJyMpMnT67+ubCwkH79+jn1WJrbFxFxP6endIKDgwHbSh2r1UpOTg4pKSkuCyYiIq5lV+EPHz6c\nrl27smvXLiIiIpg/fz4AM2bMYPTo0fTp04exY8fSvHlzh578t5Z1imOioqJISEggMTGRzp07mx3H\n61xq+XFZWRmDBg2iTZs2DB48mB9++MHEhN7jUq9ldnY2rVu3JjExkcTExEuu/JNL27dvHz179uSq\nq64iLS2NRYsWAU68Pw0TderUydiwYYNhtVqNK6+80jh8+LCZcbxeVFSUcfToUbNjeK28vDwjPz/f\nuPrqq6u3TZ8+3bjvvvuMiooKY9y4ccZTTz1lYkLvcanXMjs723jmmWdMTOW9Dh48aBQUFBiGYRiH\nDx822rZtaxw/ftzh96dp59I5f1lnZGRk9bJOqRlD+0Oc1r17d5o2bXrBtq1btzJq1Cjq169PVlaW\n3qN2utRrCXp/Ois8PJxOnToB0Lx5c6666iq2bdvm8PvTtMJ3ZFmn2MdisdCrVy8GDx7M8uXLzY7j\nE85/n8bExLB161aTE3m3WbNm0aVLF6ZPn05ZWZnZcbzS7t27KSwspHPnzg6/P3W2TB+yadMmvvji\nC6ZNm8YDDzzAoUOHzI7k9TQidZ17772XvXv3smbNGoqKipgzZ47ZkbxOWVkZw4YN47nnniMoKMjh\n96dphZ/3z2YIAAABe0lEQVScnMzOnTurfy4sLKRLly5mxfEJLVu2BCA2NpaBAwfy3nvvmZzI+yUn\nJ7Njxw4AduzYQXJyssmJvFeLFi2wWCwEBwczbtw4li1bZnYkr1JZWcmQIUPIzMxk0KBBgOPvT9MK\nX8s6XevkyZPVX5EPHz7MmjVrnD4uQn6SkpLCvHnzKC8vZ968eRqU1MDBgwcBOHPmDIsWLdLR9w4w\nDINRo0Zx9dVXc//991dvd/j96eady78qNzfXiImJMdq3b2/MnDnTzCheb8+ePUbHjh2Njh07Gr16\n9TLmzp1rdiSvc+uttxotW7Y06tWrZ7Ru3dqYN2+ecfz4cWPgwIFGRESEMWjQIKOsrMzsmF7h3GsZ\nGBhotG7d2pg7d66RmZlpxMfHG9dcc40xadIkrShzwMaNGw2LxWJ07NjR6NSpk9GpUydj1apVDr8/\nLYahSUoREX+gnbYiIn5ChS8i4idU+CIifkKFLyLiJ1T4IiJ+QoUvIuInVPgiIn7i/wEazPLwzRnk\nUQAAAABJRU5ErkJggg==\n"
      }
     ], 
     "prompt_number": 254
    }, 
    {
     "cell_type": "code", 
     "collapsed": false, 
     "input": [
      "rare = round(num_haplo*0.05)", 
      "common = round(num_haplo*0.1)", 
      "print rare,common", 
      "print sum(sfs[1:rare+1])/sum(sfs[common:])", 
      "print sum((array(h,dtype=float)/sum(h))[1:rare+1])/sum((array(h,dtype=float)/sum(h))[common:])"
     ], 
     "language": "python", 
     "outputs": [
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        "1.0"
       ]
      }, 
      {
       "output_type": "stream", 
       "stream": "stdout", 
       "text": [
        " 2.0", 
        "1.17351598174", 
        "0.869126043039"
       ]
      }
     ], 
     "prompt_number": 255
    }, 
    {
     "cell_type": "code", 
     "collapsed": true, 
     "input": [], 
     "language": "python", 
     "outputs": []
    }
   ]
  }
 ]
}